% Hybridization has become increasingly recognized to be a widespread
% phenomenon among closely related species. 
% However, it is not yet clear how important the consequences are.
% Potential consequences are adaptive introgression, introgression of neutral 
% genetic variation, reinforcement, lineage fusion, polyploidization, hybrid 
% speciation, transition to unisexual reproduction \parencite{abbott2013}. 

% The frequency of these outcomes is unknown.
% Assessing the impact of hybridization in the past is very difficult.
% Its effects become obscured with time and are not easily teased apart from other processes.
% It is much more difficult to infer models of evolution that include reticulations 
% than without due to the substantially greater number of possibilities that need 
% to be assessed.

% The degree of gene exchange that occurs in nature has been a highly debated topic. 
% Recent genomic studies have demonstrated that introgression is prevalent among 
% various groups of organisms, including fungi, vertebrates, insects, and angiosperms. 
% The consequences of introgression are diverse and depend on ecological and 
% genomic factors. 
% While it was previously believed to be mainly harmful, it is now evident that 
% introgression can be a source of genetic variation utilized in local adaptation 
% and adaptive radiation. Although our understanding of introgression as a 
% widespread phenomenon has improved, it remains uncertain how frequently it 
% occurs across different groups of organisms. 
% Ideally, determining the frequency of introgression throughout the tree of life 
% would involve analyzing clade-level genomic data systematically, without prior 
% knowledge of taxa known to hybridize in nature.

% Hybridization is a common phenomenon in nature and one which can have important 
% implications for evolution (Mallet, 2005). The role of hybridization in evolution 
% is becoming increasingly appreciated in a wide array of organisms and can have 
% varied effects on populations involved (A Genomic Perspective on Hybridization 
% and Speciation, 2016). Hybridization often results in reduced fecundity or the 
% production of unfit or sterile offspring when populations have sufficiently 
% diverged from one another. This cost has the potential to drive the evolution of 
% greater divergence between species when selection acts on traits that enhance 
% pre-mating barriers and reduce the likelihood hybridization (Servedio, 2001). 
% However, hybridization can also serve as a source of novel, adaptive genetic variation 
% through introduction of novel genetic variants or combinations thereof through 
% introgression between populations (Oziolor et al., 2019; Pardo-Diaz et al., 2012). 
% It has even been suggested that selection could maintain some degree of compatibility
% between species as a result of occasional introgression of adaptive genetic 
% variation (Servedio & Hermisson, 2020). In rare instances hybridization may 
% result in the emergence of new species from hybrid populations (Schumer et al., 2014). 
% If barriers to hybridization are not sufficiently strong, hybridization could 
% lead to the erosion of barriers between populations (Chafin et al., 2019). 
% Given that it can have important consequences in the evolution of lineages, 
% inference of historic introgression is important for a complete understanding of 
% a lineage’s evolutionary history and for understanding the consequences of 
% contemporary hybridization. 

% North American toads in the genus Anaxyrus have been the subject of many studies 
% of contemporary hybridization but we know very little about the historical context 
% preceding it. Many species pairs have been found to hybridize naturally or in the 
% laboratory (Blair, 1972; Green, 1996). Several attributes make toads appealing 
% for the study of hybridization. They are very abundant and conspicuous, and their 
% primary sexual signal – advertisement call— can be readily measured and quantified. 
% There has been success with rearing them in the lab and they produce very large clutch sizes. 
% Sex determination in toads is unique among species with historical prominence in
% hybridization research. Many bufonid toads are known to possess homomorphic sex chromosomes. 
% There have also been several transitions between XY and ZW sex determination. 
% This is interesting because many studies have found evidence for an important 
% role for sex chromosome in the evolution of reproductive incompatibility (Coyne & Orr, 2004). 
% But most studies have focused on species with heteromorphic sex chromosomes 
% where hybridization occurs between species with XY sex determination systems.

% While North American toads are an attractive system for gaining more understanding 
% of the process of speciation and evolution of reproductive incompatibility, 
% we currently lack important historical context for hybridization in these toads. 
% Foremost is uncertainty about the evolutionary relationships among species and populations. 
% It is not clear how closely related hybridizing species are to one another and whether 
% they are each other’s closest living relatives. A number of studies have estimated their 
% phylogenetic relationships, but there is some disagreement among these studies 
% (Fontenot et al., 2011; Graybeal, 1997; Masta et al., 2002; Pramuk et al., 2008; Pyron et al., 2011). 
% Particularly in the americanus group which is composed of 7 currently recognized species. 
% The inconsistencies in inferred relationships could be due to many possible 
% sources of error known to affect phylogenetic inference (Som, 2014). 
% Furthermore, some studies only included a single sample from currently 
% recognized species, and thus were not able to confirm the monophyly of these species 
% (Pyron & Wiens, 2011). In studies that did sample multiple individuals per
% species, some, like A. americanus and A. fowleri, were inferred to be polyphyletic (Fontenot et al., 2011).
% The inconsistencies in inferred relationships among Anaxyrus species could be caused by 
% real biological processes such as incomplete lineage sorting or introgressive 
% hybridization (Degnan & Rosenberg, 2009; Kubatko & Degnan, 2007).
% Incomplete lineage sorting or hybridization could result in individual gene 
% trees that differ from the true species history. Graybeal, 1997 and Masta et al., 2002 used only mitochondrial sequence data which may not be consistent with the evolutionary history of the nuclear genome or that of the species due to incomplete lineage sorting or mitochondrial introgression. Fontenot et al., 2011, Pramuk et al., 2008, and Pyron & Wiens, 2011 used concatenated alignments of mtDNA and a small number of nuclear loci. However, concatenated analyses can be misleading when there is high discordance among gene trees due to incomplete lineage sorting (Kubatko & Degnan, 2007; Roch & Steel, 2015). Given the large number of putative hybrids observed between contemporary populations of toads, historic hybridization resulting in a significant amount of gene tree discordance seems highly plausible. Quantifying how much discordance is caused by hybridization would also be important context for understanding contemporary hybridization and the evolution of reproductive barriers between species. A history of hybridization and admixture among ancestral lineages could play an important role in the degree of reproductive isolation that exists between species of toads today. 
% Knowing when species have diverged is also important context for understanding 
% contemporary hybridization and the evolution of reproductive isolation. 
% It is still unknown how time since divergence factors into the evolution of 
% reproductive incompatibility. Knowledge of divergence times can also shed light 
% on drivers of speciation. For example, divergence times that are shared among 
% different pairs of species are particularly interesting in that they suggest the 
% processes that have driven diversification may have been shared among taxa.

% I propose to obtain genome wide sequence data from samples representing a 
% large portion of the ranges of most described species in the genus Anaxyrus. 
% With these data, I will test the following hypotheses: 1) recent biogeographic 
% processes have reduced or eliminated gene flow between populations within species; 
% 2) contemporary hybridization occurs between species with overlapping or abutting ranges; 
% 3) ancient introgression has occurred between lineages; 
% 4) large-scale biogeographic processes have driven diversification in Anaxyrus; 
% and 5) the current taxonomy of Anaxyrus is not consistent with evolutionary 
% relationships among populations within named groups. 

\section{Introduction}
% Many factors influence diversification.
% - Environment is really important
%   - Changes connectivity
%     - Creates opportunity for divergence
%       - Adaptive or neutral
%   - Drives adaptive change
%   - Influences population size which in turn affects drift
% - Degree of isolation is influenced by environment
% - Indirectly related to environment are the effects of introgression.
%   - Enabled by environmental changes that effect connnectivity.
%   - Itself can have important effects on evolution.
% - Understanding the interplay of all of these factors is critical for understanding
% diversification of organisms.

Many factors are understood to be important in driving and shaping the 
diversification and evolutionary history of organisms. 
Chief among them is the interplay between climatic conditions and geologic processes.
Changes in these environmental variables can alter the distributions of organisms
and result in changes in the connectivity of populations.
Disconnected populations may undergo genetic divergence from one another due 
to adaptive evolution in response to changing abiotic or biotic conditions. 
Or they might simply diverge via neutral evolution driven by the effects of drift.
Environmental changes can also reconnect previously isolated populations resulting
in hybridization and gene flow, another very important process shaping patterns of diversity.
Understanding the interplay of all of these factors is critical for understanding
the evolutionary history of organisms.
A critical step to understanding these processes is to obtain an accurate 
reconstruction  of the evolutionary history of organisms. 


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Introducing anaxyrus
The North American toads in the genus \anaxyrus are a group of organisms with a 
poorly understood evolutionary history. 
Although, not for lack of trying.
Multiple studies of the evolutionary relationships among species in the genus have produced
conflicting results \parencite{fontenot2011,graybeal1997,masta2002,pramuk2007,pyron2011,portik2023}.
Particularly within the \americanus group composed of \amer, \baxteri, \fowl, 
\hemiophrys, \houstonensis, \microscaphus, \terr, and \wood.
Two phylogenetic studies inferred trees with \fowl forming a polytomy making 
them inconsistent with the current taxonomy of \anaxyrus \parencite{fontenot2011,masta2002}.
These conflicting results could be due to methodological differences such as
the species included, the number of individuals of each species sequenced, inference 
methods used, or the sequenced loci. 
But the differences in inferred relationships could also result from real
biological processes. 
Incomplete lineage sorting is one potential source of discordance among datasets 
which include different loci that arises from real biological processes and 
impacts phylogenetic inference \parencite{kubatko2007}. 
Incomplete lineage sorting could also produce the polytypic relationship among
\fowl. 



% Move to another paragraph towards the end
% Only two studies have specifically examined patterns of genetic diversity within
% species and included multiple samples per species \parencite{fontenot2011,masta2003}.
% \cite{fontenot2011} using AFLPs found limited evidence...
% \cite{masta2003} found two distinct clades of \wood
% Found polyphyly with \fowl.
% Several subspecies have been recognized in the past raising the possibility of
% significant differentiation within species or possibly even unrecognized diversity.
% A. americanus charlsmithi
% A. americanus americanus
% A. americanus copei
% A. woodhousii velatus
% A. woodhousii australis
% A. woodhousii woodhousii

% Diversification in toads is poorly understood.
% - Inferred relationships inconsistent
%   - Go over all of the studies and conflicts
% - Mitochondrial tree has polytomies
% - What are patterns of diversity within species? 
% - Is existing taxonomy adequate?


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Potential for introgression in anaxyrus
Gene flow is another potential source of discordance among genes which could  
drive the differences in inferred relationships among studies using different 
loci and could also produce the pattern seen in \fowl \parencite{degnan2009}. 
While incomplete lineage sorting is very likely to have impacted patterns of genetic  
variation in \anaxyrus, gene flow due to hybridization is a distinct possibility as well.
There are numerous reports of natural hybridization between several different 
species of \anaxyrus \parencite{green1996}. 
A study of allozyme variation across a hybrid zone between \amer and 
\hemiophrys revealed introgression taking place across a more than 50km wide hybrid zone.
In the previous chapter I presented the results of a study on the hybrid zone
between \amer and \terr.
\cite{meacham1962} presented compelling evidence on the basis of morphological 
variation for the existence of a hybrid zone between \fowl and \wood in East Texas.
Furthermore, numerous laboratory crosses have been performed between pairs of \anaxyrus 
species with currently overlapping distributions \parencite{blair1972,blair1963}.
Some of which produce viable and fertile backcross progeny \parencite{blair1972,blair1963}.
These studies suggest that gene flow could very easily have played a role in shaping patterns
of diversity in \anaxyrus. 
However, they provide only a snapshot in time with no indication of the long term 
consequences.
There are many potential consequences of hybridization such as adaptive introgression, 
introgression of neutral genetic variation, reinforcement, lineage fusion, 
polyploidization, hybrid speciation, or transition to unisexual reproduction \parencite{abbott2013}. 
Inference of past introgression is an important starting point for exploring these
outcomes yet it remains a challenging problem. 
The network structure of phylogenetic networks are far less tractable to infer than the 
more simple bifurcating phylogenies for which there has been extensive method development.
There has been some recent work to overcome this challenge as well as increased 
feasibility of obtaining appropriate genome wide datasets to investigate past 
gene flow.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Hybridization is useful but needs context
Apart from the significant evolutionary implications of hybridization which 
need to be understood, it also presents a valuable opportunity for investigating 
the mechanisms that drive divergence and the evolution of reproductive
incompatibility \parencite{rieseberg1999}. 
Many generations of backcrossing within hybrid zones can produce a large number
of highly recombinant genomes that allow for the observation of many possible 
hybrid genotypes under natural conditions in order to identify advantageous or
disadvantageous hybrid genotypes. 
In most species it is not feasible to produce such a large number of highly  
recombinant offspring in order to make such observations.
The evolutionary history of hybridizing species is important context to have 
when studying patterns of introgression within hybrid zones. 
Context such as the phylogenetic relationship of hybridizing species relative to
other closely related species, the amount of genetic differentiation, 
the time since divergence and by extension the biogeographic 
processes driving initial divergence and subsequent secondary contact in cases 
of allopatric divergence.
This important context is currently missing for \anaxyrus which limits the
inferences that can be made regarding hybrid zones in this genus. 


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Effect of environment on toads
Ultimately, environmental change is what leads many populations divergence and 
and may lead to subsequent or concurrent gene flow.
Therefore, environmental variables are also important context for making 
inferences from hybrid zones in addition to understanding the process of 
diversification more generally.
To date, there have not been any studies conducted to understand how the 
environment as driven diversification in North American toads. 
North America has had a very complex geologic and climatic history \parencite{lyman2022}.
The effects of which are often clade specific \parencite{nunez2023}.
But large scale environmental changes can impact multiple species simultaneously \parencite{oaks2019}.
There has been recent development in methods to infer these events \parencite{oaks2019,oaks2022}. %Cite hickerson
The identification of multiple pairs of lineages that underwent divergence at 
the same time could provide evidence about environmental changes driving 
diversification. %Citation neeed
Present day population structure could also provide further understanding by 
revealing environmental factors that reduce gene flow assuming the biological 
limits of present day species have not evolved dramatically from the ancestral 
condition.

% Say something about these events and how they are useful
% - We don't know what factors have driven diversification
% - Presumably they would be climatic.
% - Cite studies on the effect of climate for North American taxa
% - Environment can affect whole communities simultaneously.
% - Are there environmental factors influencing within species genetic variation?


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Goals of this study
In this study, I investigate the evolutionary history of North American
toads in the genus \anaxyrus using genome wide sequence data. 
For this I obtained restriction enzyme-associated DNA sequence (RADseq) data
from 12 species of \anaxyrus including dense sampling representing a 
large portion of the ranges of \amer, \fowl, \terr, and \wood.
With these data I infer evolutionary relationships using maximum likelihood analysis  
of a concatenated dataset of many broadly distributed samples in addition to 
using a multispecies coalescent analysis of a subset of the data.
I also test for the presence of shared divergence times which might suggest
\anaxyrus diversification has been driven by the same environmental changes
and also estimate the absolute timing of all divergences within the genus.
With the robust estimate of phylogenetic relationships among \anaxyrus species, 
I test for the presence of ongoing and historic introgression among \anaxyrus species.
In order to identify the types of environmental factors that might have played
a role in isolating populations that would eventually diverge as species,
I investigate population structure within a subset of \anaxyrus species.
Finally, I estimate proportions admixture between \fowl and \wood to test
the hypothesis that these species form a hybrid zone in the central United  
States where their ranges meet.

% 1) recent biogeographic 
% processes have reduced or eliminated gene flow between populations within species; 
% 2) contemporary hybridization occurs between species with overlapping or abutting ranges; 
% 3) ancient introgression has occurred between lineages; 
% 4) large-scale biogeographic processes have driven diversification in Anaxyrus; 
% and 5) the current taxonomy of Anaxyrus is not consistent with evolutionary 
% relationships among populations within named groups. 


\section{Methods}
\subsection{Sampling and DNA Isolation}
% TODO: How many samples
I obtained tissue samples from museum tissue collections as well as from individuals   
I collected from 2017 to 2020. I selected samples to represent as much
of the range of each species of \textit{Anaxyrus} as possible.
I also included one \textit{Rhinella marina} and one \textit{incilius nebulifer} 
for as outgroups for phylogenetic analyses.

I isolated DNA from tissues by first lysing a piece of tissue approximately 
the size of a grain of rice in 300 \uL\ of a solution of 10mM Tris-HCL, 10mM EDTA, 
1\% SDS (w/v), and nuclease free water along with 6 mg Proteinase K that was 
incubated for 4-16 hours at 55\degree C in a 1.5 mL microcentrifuge tube.  
To purify the DNA and separate it from the lysis product, I mixed the lysis 
product with a 2X volume of SPRI bead solution containing 1 mM EDTA,  
10 mM Tris-HCl, 1 M NaCl, 0.275\% Tween-20 (v/v), 18\% PEG 8000 (w/v), 
2\% Sera-Mag SpeedBeads (GE Healthcare PN 65152105050250) (v/v), and nuclease free water.
I then incubated the samples at room temperature for 5 minutes, placed the 
beads on a magnetic rack, and discarded the supernatant once the beads had collected
on the side of the tube.  
I then performed two ethanol washes by adding 1 mL of 70\% ETOH to the beads
while still placed in the magnet stand and allowing it to stand for 5 minutes
before discarding the ethanol. 
After removing all ethanol from the second wash, I removed the tube from the magnet 
stand and allowed the sample to dry for 1 minute before mixing the beads with 100 \uL\ of 
TLE solution containing 10 mM Tris-HCL, 0.1 mm EDTA, and nuclease free water.
After allowing the bead mixture to stand at room temperature for 5 minutes I returned
the beads to the magnet stand, pipetted all of the TLE solution into another 
microcentrifuge tube, and discarded the beads. I quantified DNA with a Qubit
fluorometer (Life Technologies, USA) and diluted samples with TLE solution to 
bring the concentration to 20 ng/\uL.

\subsection{RADseq Library Preparation}
I prepared RADseq libraries using the 2RAD approach outlined by \cite{bayona-vasquez2019}. 
On 96 well plates, I ligated 100 ng of sample DNA in 15 \uL\ of a solution with 
1X CutSmart Buffer (New England Biolabs, USA; NEB), 10 units of XbaI,
10 units of EcoRI, 0.33 \uM\ XbaI compatible adapter, 0.33 \uM\ EcoRI compatible adapter,
and nuclease free water with a 1 hour incubation at 37\degree C. 
I then immediately added 5 \uL\ of a solution with 1X Ligase Buffer (NEB),
0.75 mM ATP (NEB), 100 units DNA Ligase (NEB), and nuclease free water 
and incubated at 22\degree C for 20 min and 37\degree C for 10 min for two cycles, 
followed by 80\degree C for 20 min to stop enzyme activity.
For each 96 well plate, I pooled 10 \uL\ of each sample and split this pool 
equally between two microcentrifuge tubes.
I purified each pool of libraries with a 1X volume of SpeedBead solution followed 
by two ethanol washes as described in the previous section except that the DNA 
was resuspended in 25 \uL\ of TLE solution. 

In order to be able to detect and remove PCR duplicates, I performed a single   
cycle of PCR with the iTru5-8N primer which adds a random 8 nucleotide barcode to 
each library construct.  
For each plate, I prepared four PCR reactions with a total volume of 
50 \uL\ containing 1X Kapa Hifi Buffer (Kapa Biosystems, USA; Kapa),
0.3 \uM\ iTru5-8N Primer, 0.3 mM dNTP, 1 unit Kapa HiFi DNA Polymerase,
10 \uL\ of purified ligation product, and nuclease free water.
I ran reactions through a single cycle of PCR on a thermocycler at 98\degree C for 2 min, 
60\degree C for 30 s, and 72\degree C for 5 min. 
I pooled all of the PCR products for a plate into a single tube and purified the
libraries with a 2X volume of SpeedBead solution as described before and 
resuspended in 25 \uL\ TLE.
I added the remaining adapter and index sequences unique to each plate with four PCR
reactions with a total volume of 50 \uL\ containing 1X Kapa Hifi (Kapa),
0.3 \uM\ iTru7 Primer, 0.3 \uM\ P5 Primer, 0.3 mM dNTP, 1 unit of Kapa Hifi DNA Polymerase (Kapa),
10 \uL\ purified iTru5-8N PCR product, and nuclease free water.
I ran reactions on a thermocycler with an initial denaturation at 98\degree C for 2 min, 
followed by 6 cycles of 98\degree C for 20 s, 60\degree C for 15 s, 72\degree C 
for 30 s and a final extension of 72\degree C for 5 min.
I pooled all of the PCR products for a plate into a single tube and purified the
product with a 2X volume of SpeedBead solution as described before and 
resuspended in 45 \uL\ TLE.

I size selected the library DNA from each plate in the range of 450-650 base pairs using
a BluePippin (Sage Science, USA) with a 1.5\% dye free gel with internal R2 standards. 
To increase the final DNA concentrations I prepared four PCR reactions for each 
plate with 1X Kapa Hifi (Kapa), 0.3 \uM\ P5 Primer, 0.3 \uM\ P7 Primer, 0.3 mM dNTP, 
1 unit of Kapa HiFi DNA Polymerase (Kapa), 10 \uL\ size selected DNA, and 
nuclease free water and used the same thermocycling conditions as the previous
(P5-iTru7) amplification.
I pooled all of the PCR products for a plate into a single tube and purified 
the product with a 2X volume of SpeedBead solution as before and resuspended in 20 \uL\ TLE. 
I quantified the DNA concentration for each plate with a Qubit fluorometer 
(Life Technologies, USA) then pooled each plate in equimolar amounts relative 
to the number of samples on the plate and diluted the pooled DNA to 5 nM with
TLE solution. 
The pooled libraries were pooled with other projects and sequenced on an Illumina 
HiSeqX by Novogene (China) to obtain paired end, 150 base pair sequences. 

\subsection{Phylogenetic Data Processing}
To produce alignments for phylogenetic analysis, I first I demultiplexed the 
iTru7 indexes using the \processradtags command from \stacks v2.6.4 
\parencites{rochette2019} and allowed for two mismatches for rescuing reads.
I removed PCR duplicates using the the \clonefilter command from \stacks.
To demultiplex individual samples I used \pyrad v0.9.90 and allowed for one 
mismatch for rescuing reads. 
I assembled and aligned reads with \pyrad using default parameters and a 
clustering threshold of 0.8. 
Using \pyrad, I filtered loci not present in at least 75\% of samples 
and filtered samples with fewer than 200 loci.

\subsection{Maximum Likelihood}
Phylogenetic methods that do not account for incomplete lineage sorting  
do not perform well with data impacted by this process.
However, methods that do account for incomplete lineage sorting are far more 
computationally demanding.
As a result, these methods cannot be performed with a large number of samples.
I therefore conducted conducted maximum likelihood phylogenetic inference in  
order to infer a phylogeny with all of the sequenced samples and to be able  
to identify samples that may be problematic for other methods due to recent 
admixture or data quality. 
I conducted the maximum likelihood phylogenetic inference with \iqtree 
v1.6.12 \parencite{nguyen2015} with the \pyrad alignment as input in order. 
I ran \iqtree with 1000 ultrafast bootstrap replicates \parencite{hoang2018}
under the GTR substitution model.

\subsection{Multispecies Coalescent}
In order to account for incomplete lineage sorting in the inference of phylogenetic
relationships and to infer shared divergence times, I used the program \phycoeval.
I selected a subset of up to four samples from each species due to the 
infeasible run times for \phycoeval with greater numbers of samples (see table 1).
I excluded sample 006 from consideration due it having an anomalous   
position in the maximum likelihood tree.
I used \pyrad to filter loci not present in at least 75\% of samples. 
Using a custom script I filtered the phylip alignment file produced by \pyrad 
to exclude sites with more than two  characters and output the filtered alignment 
to nexus format with a biallelic character encoding. 
I ran \phycoeval with state frequencies fixed at 0.5.
I set the mutation rate equal to one so that divergence times are in units of 
expected substitutions per site. 
I set the prior on the age of the root as an exponential distribution with a mean
of 0.01.
I ran \phycoeval with the assumption of a single effective population size
shared across all of the branches of the tree.
The prior on the effective population size was a gamma prior with a shape of
four and mean of 0.0005
I ran five independent MCMC chains for 10,000 generations, sampling every 10 
generations.
Each chain was started with a comb tree topology with all branches sharing the
same divergence time. 
I summarized the posterior sample of tree topologies and parameters using 
\sumphycoeval.
To assess convergence and mixing, I used \sumphycoeval to calculate the
potential scale reduction factor (PSRF) and the effective sample size (ESS).
I discarded the first 100 samples from each chain as burnin.
I used \sumphycoeval to rescale the branch lengths of the maximum a posteriori 
(MAP) tree produced by \sumphycoeval so that the posterior mean root age was 
16.5 million years ago based on the estimate of \cite{feng2017}.

\subsection{Introgression}
In order to test for introgression between species of \anaxyrus I used the 
program \dsuite v0.5r50 \parencite{malinsky2021} to compute the $f$-branch 
statistic for each pair of \anaxyrus species for which the statistic 
can be calculated \parencite{reich2009,malinsky2018}. 
I used \pyrad to filter all loci that were not found in at least 50\% of the 
samples that passed filtering and excluded one \fowl sample 
% (sample 006 from \cref{tab:phylo}) which falls outside of the \fowl clade 
inferred by \iqtree.
For the input tree topology required to run \dsuite, I used the topology inferred
by \phycoeval and I specified \nebulifer as the outgroup species.
I ran the \dsuite Dtrios command to compute Patterson's the $f4$-ratio
statistic for all possible trios with 20 block-jackknife replicates.
I then ran the Fbranch command from \dsuite to compute the $f$-branch statistics 
from the computed $f4$-ratio statistics. 
I plotted the $f$-branch statistics with \dtools v0.1 which is packaged with
the \dsuite program \parencite{malinsky2021}. 

Say something about how f-branch takes into account correlation among branches
% Add the \structure methods for \fowl and \wood here 

\subsection{Population Structure Data Processing}
I processed reads differently for the analysis of population structure
following PCR duplicate filtering. 
I demultiplexed individual samples, trimmed adapter sequence, and filtered 
reads with low quality scores as well as reads with any uncalled bases using the 
\processradtags command and allowed for the rescue of restriction site sequence 
as well as barcodes with up to two mismatches.  
I allowed for 14 mismatches between alleles within, as well as between individuals
(M and n parameters). This is equivalent to a sequence similarity threshold of   
90\% for the 140 bp length of reads post trimming. 
I also allowed for up to 7 gaps between alleles within and between individuals.
I used the \populations command from \stacks to filter loci missing in more than   
5\% of individuals, filter all sites with minor allele counts less than 3, filter 
any individuals with more than 90\% missing loci, and randomly sample a single
SNP from each locus.

% To remove PCR duplicates, I used the \clonefilter command from \stacks.
% I demultiplexed inline sample barcodes, trimmed adapter sequence, and filtered 
% reads with low quality scores as well as reads with any uncalled bases using the  
% \processradtags command again and allowed for the rescue of restriction site sequence 
% as well as barcodes with up to two mismatches.  


\subsection{Population Structure}
To investigate population structure within \amer, \fowl, \terr, and \wood , I used the demultiplexed
and de-cloned reads used for the phylogenetic analyses for producing alignments. 
I assembled and aligned these reads using \stacks for each species separately.
I allowed for 7 mismatches between alleles within, as well as between individuals
(M and n parameters). This is equivalent to a sequence similarity threshold of   
95\% for the 140 bp length of reads post trimming. 
I also allowed for up to 7 gaps between alleles within and between individuals.
I used the \populations command from \stacks to filter loci missing from more than
5\% of samples, filter all sites with minor allele counts less than 3, filter 
any individuals with more than 90\% missing loci and to randomly sample a single
site per locus.

I ran the program \structure v2.3.4 \parencite{pritchard2000} for each species  
separately using the admixture model in order to cluster individuals and estimate
ancestry proportions for each individual. 
I ran \structure under four different models differing in the number of 
populations assumed (K parameter), with the parameter ranging from 1-4.
I ran 10 iterations of \structure for each value of K for a total of 100,000
steps and burnin of 50,000 for each iteration.
I used the R package \pophelper v2.3.1 \parencite{francis2017} to combine
iterations for each value of K and to select the model producing the largest
$\Delta$K which is the the model that has the greatest increase in likelihood 
score from the previous model having one fewer populations as described by \parencite{evanno2005}.
I also investigated population structure with a non-parametric approach, using
principle component analysis (PCA) implemented in the R package \adegenet
\adegenet v2.1.10 \parencite{jombart2008}. 

\subsection{Recent \fowl x \wood hybridization}

% I ran structure with all of the \amer group samples together to ensure that 
% none of the samples have been affected by recent admixture before inferring
% population structure within each species.
% Each structure analysis was run with assemblies produced by \stacks.
% For the analysis of all \amer group samples,
% I dropped 16 samples 
% I filtered...
% Dropped sites with minor allele counts greater than three.
% I excluded samples...
% Ran with K 1-4
% 10 independent runs
% 100,000 generations and a burnin of 50,000 samples



\section{Results}

\subsection{Assembly and alignment with \pyrad}
A total of 436,265,266 reads were obtained for all samples. After filtering low
quality reads and reads without restriction site sequence, 435,650,926 total reads 
remained for assembly.
The number of filtered reads per individual was highly variable with a mean of 
4,538,030 (sd=3,619,076).
Prior to filtering there were 171,174 loci total loci which was reduced to  
659 after filtering loci not present in at least 75\% of samples and filtering
?? samples which had fewer than 200 loci (Table 1).
Mean sequence read coverage of the loci passing filter was 54x.
The final alignment contained a total of 184,453 sites with 20,361 SNPs with 
14.96\% of sites and 14.71\% of SNPs missing.

\subsection{Maximum Likelihood Phylogeny}
The full majority rule consensus tree inferred by \iqtree is presented in \cref{fig:iqtree}. 
All species were inferred as a single monophyletic group with the exception 
of \fowl. 
A single \fowl sample (sample 006) does not form a monophyletic group with other 
\fowl samples but is instead sister to the the branch containing \wood and \fowl samples.
A representation of the tree inferred by \iqtree with the tips within
species specific clades collapsed is presented in \cref{fig:iqtree-collapsed}. 
Each species specific clade for which there are at least two representatives
samples all have ultrafast bootstrap support values of 100\%.
All branches below the the level of the species specific clades have ultrafast 
bootstrap support values ranging from 70-100\% with the majority being 100\%.
The most basal internal branch of the tree, marking the split between most of \anaxyrus
and \punctatus along with the outgroup \nebulifer has an ultrafast bootstrap 
support value of 99\%.
The sister branch to to \terr, which contains the spurious \fowl sample (sample 006)
and the clade containing \fowl and \wood, has an ultrafast bootstrap support 
value of 96\%.
The lowest ultrafast bootstrap support value is found on the branch sister 
to the \cognatus/\speciosus clade with a value of only 70\%.

\subsection{Coalescent Phylogeny}
The maximum a posteriori (MAP) tree inferred under the multispecies coalescent
model using \phycoeval has a topology differs from the maximum likelihood topology 
inferred by \iqtree \cref{fig:phycoeval}.
The MAP tree produced by \phycoeval does not have any shared divergence times 
among any  of the 10 internal nodes of the tree.
The frequency of topologies in the posterior sample that have 10 independent 
divergence times is 0.5.
The next most frequent topology in the posterior are topologies with a single 
shared divergence time and nine independent divergences and occur with a frequency
of 0.24.
One major difference between the maximum likelihood tree inferred by \iqtree and 
the MAP tree inferred by \phycoeval is that the MAP tree has one multifurcation. 
This multifurcation happens at the ancestor of the \quercicus, \speciosus/\cognatus,
and \amer group lineages.
However, this node has a low posterior probability of only 0.51. 
All other branches in the MAP tree have high posterior probabilities of 0.98 or more. 
Most divergence events within \anaxyrus have occurred in the past 3.5 million 
years and most diversification within the \amer group is less than 2.5   
million years old.

\subsection{Introgression}
I used the program \dsuite to compute the \fbranch statistic which is an  
estimate of excess allele sharing between species pairs that is not due to 
incomplete lineage sorting. 
I used the species tree topology produced by \phycoeval for estimating 
the \fbranch statistics. 
The \fbranch estimates for each species pair are presented with a heat map in 
figure \cref{fig:dsuite}.
Most \fbranch estimates produced by \dsuite were zero or very near zero.
Only 24 out of 112 \fbranch estimates were greater than 0 and 11 of those were 
greater than 0.05 \cref{fig:dsuite}.
\amer and \wood had the largest number of estimates greater than
zero associated with them with nearly every pairwise comparison greater than 0
\cref{fig:dsuite}. 
The highest \fbranch statistic values are between \amer and two other 
species: \hemiophrys (0.24) and \baxteri (0.22) \cref{fig:dsuite}.
The values associated with \wood are appreciably lower with none exceeding 
0.1 \cref{fig:dsuite}.
The highest being between \amer and \wood with a value of 0.098 \cref{fig:dsuite}.
The \wood \fbranch values for \baxteri and \hemiophrys are 0.082 and 0.086 
respectively \cref{fig:dsuite}.
The \fbranch value between \wood and \microscaphus is 0.05.
Finally, the smallest \wood \fbranch values are in the tests with \cognatus and 
\speciosus at 0.023 and 0.029 respectively.


\subsection{Population Structure}

\subsection{Hybridization between \fowl and \wood}


\section{Discussion}

\subsection{Phylogenetic relationships}
% Maximum likelihood tree
The maximum likelihood tree inferred by \iqtree \cref{fig:iqtree,fig:iqtree-collapsed} 
differs from trees inferred in previous studies of the relationships 
among \anaxyrus \parencite{fontenot2011,graybeal1997,masta2002,pramuk2007,pyron2011,portik2023}.
Even among these previous studies there has been a great deal of inconsistency in
the inferred relationships except in for the position of a few taxa.
As in all previous studies, the maximum likelihood tree inferred in this 
study places \punctatus sister to all other \anaxyrus.
I also found the \americanus group to be monophyletic with \microscaphus sister  
to all other \americanus group species which is consistent with most previous studies.
Two previous studies have inferred trees which do not place \fowl samples
into a single monophyletic group \parencite{masta2002,fontenot2011}.
A single \fowl sample included in this study does not fall within a monophyletic 
group with the remaining \fowl samples but is instead sister to the clade 
containing all \fowl and \wood samples \cref{fig:iqtree-collapsed}.

% Possible causes
All of these studies have included different species, individuals, and loci, and 
also used different methods for alignment and phylogenetic inference. 
These differences in study design could result in the observed topology differences. 
The choice of locus in particular has a high likelihood of being the cause of these differences. 
Due to incomplete lineage sorting, the true histories of each gene may in fact 
differ from one another and not reflect the history of the species \parencite{kingman1982}. 
The practice of concatenating multiple loci as all of the previous studies of
\anaxyrus evolutionary relationships have done, can produce erroneous trees  
with high statistical support \parencite{kubatko2007}.
Despite the inappropriateness of concatenated analysis with genome-wide data,
it was reassuring to find that all but one individual clustered with members
of it's own species. 
\anaxyrus can be challenging to identify, particularly in a preserved state. 
The maximum likelihood tree does not suggest that any samples in the dataset 
have been misidentified which could be problematic for other analyses.

% Coalescent
To account for incomplete lineage sorting, I also inferred phylogenetic     
relationships among \anaxyrus species using the multispecies coalescent 
method \phycoeval along with a subset of individuals used for the maximum likelihood 
tree due to increased computational demands of multispecies coalescent methods. 
The topology of the \phycoeval tree is substantially different from the  
maximum likelihood tree inferred in this study as well as trees from previous 
studies \cref{fig:phycoeval} \parencite{fontenot2011,graybeal1997,masta2002,pramuk2007,pyron2011,portik2023}. 
Unlike in any previous study or in the maximum likelihood tree, \amer and \terr are 
placed sister to one another, whereas in all other trees it has had closer 
affinity to the \hemiophrys/\baxteri clade \cref{fig:iqtree-collapsed} \parencite{pyron2011,portik2023}.
In the \phycoeval tree, the \hemiophrys/\baxteri clade is instead sister
to the \amer/\fowl/\terr/\wood clade.

An unusual feature of \phycoeval is that it can allow for multifurcations in  
inferred topologies \parencite{oaks2022}.
This feature proved relevant for in this study as the inferred tree included
one multifurcation at the ancestral node of \quercicus, the \cognatus/\speciosus
clade, and the \americanus group. 
Previous studies have produced trees with quite short internode branches at this
part of the tree as did the \iqtree analysis in this study which is somewhat 
consistent with this. 
These methods can only produce bifurcations and thus would force any true 
multifurcation into bifurcations and estimate some branch length between them
which would be expected to be short. 
In the \phycoeval tree, the posterior probability of this split is low (0.51) so 
it may not be a perfect representation of the history of these lineages \cref{fig:phycoeval}.
More data may be necessary to have full resolution in this part of the tree.
But it is clear that these three lineages diverged at least in very rapid succession
if not simultaneously. 
% But I don't know of any significant implications these alternative scenarios would
% have for our understanding of \anaxyrus evolution.

% \quercicus forms a polytomy with the \cognatus/\speciosus clade and the \americanus
% clade although this node has low posterior probability support (0.51).
% The internode branch lengths in this part of the tree are quite short in the 
% maximum likelihood tree. 
% The placement of \quercicus relative to the \cognatus/\speciosus clade and the 
% \americanus clade has been fairly inconsistent across other studies but these
% other studies have also found short internode branch lengths.
% A polytomy may be a good representation of this node.
% A larger amount of data may be necessary to more confidently determine the 
% relationships.


\subsection{Divergence Time}
Only three previous studies have produced estimates for age of the \anaxyrus
lineage \cite{frazao2015,feng2017,portik2023}.
The \cite{frazao2015} phylogeny places \incilius sister to \rhinella rather 
than \incilius which is not supported by most recent studies making 
the approximately 23 mya estimate for the origin of the genus questionable \cite{feng2017,portik2023,pyron2011}. 
\cite{portik2023} estimate the split between \anaxyrus and \incilius to be
20.3 mya (95\% HPD: 17.8-22.5) whereas \cite{feng2017} estimate a much earlier 
age of 16.5 mya (95\% CI: 14.0-19.4).
The dataset from \cite{feng2017} included near complete coverage from 95 nuclear loci 
whereas the \cite{portik2023} has a higher degree of missing data (95\%)
and includes both mitochondrial as well as nuclear loci. 
For these reasons I consider the \cite{feng2017} estimate to be the most reliable
and chose it for the rescaling the branch lengths of the \phycoeval tree.

Scaling the root of the \phycoeval tree I estimated with the \cite{feng2017} estiate
puts the time since the most recent common ancestor (MRCA) of extant \anaxyrus 
some time between 11.9 mya when \punctatus diverged from other \anaxyrus and 16.5 mya
when \anaxyrus split form \incilius \cref{fig:phycoeval}.
This range is is not inconsistent with the estimate of 12.3 mya (95\% CI: 9.7-15.2) 
made by \cite{feng2017}.
But it would suggests that it must have happened almost immediately before the 
split leading to \punctatus. 
\cite{portik2023} estimate the age of MRCA of \anaxyrus to be approximately halfway 
between the 14.7 mya \punctatus split and the 20.3 mya split with \incilius at 16.7 mya. 
This study and the previous ones, have a high degree of uncertainty around the 
ages of these basal splits in the \anaxyrus tree. 
But it seems that that the split between \incilius and \anaxyrus likely happened 
somewhere around the start or just before the middle of the Miocene epoch.
The MRCA of \anaxyrus and the split between the \boreas group with a Western 
distribution, likely occurred prior to the middle of the Miocene. 
My estimate for the split between \punctatus and other \anaxyrus would be 
right at the middle of the Miocene at a time when both precipitation and 
temperature underwent a decline in the North American interior and there was
expansion of grasslands \parencite{morales-garcia2020}.
The timing of the multifurcation of the \quercicus, \cognatus/\speciosus, and 
\americanus group lineages coincides with a previously identified shift in the
ecomorphology of ungulate mammals inhabiting North America \parencite{morales-garcia2020}. 

I estimate that diversification of the \americanus group has all happened in the 
past 3.4 million years. This accounts for a large portion of the diversity of  
\anaxyrus and includes two additional un-sampled species which other studies  
have found to be nested within this clades \cite{portik2023,pyron2011}. 
Those being \houstonensis and \californicus.
This means that most diversification within \anaxyrus took place just before  
and during the Pleistocene 2.58 million to 11,700 years ago. 
This is a period marked by extreme climatic variation and repeated glacial cycles 
that transformed the climate and geography of the North American continent \parencite{holman1995,holman2003}.
Surprisingly, there is no evidence from the \phycoeval analysis that any single 
one of these cycles was a driver of multiple diversification events and instead 
each event occurred independently during this period of \anaxyrus evolution.

\subsection{Hybridization and Introgression}
There are numerous reports of hybridization among many different pairs of 
\anaxyrus species. 
However, the consequences of this hybridization are largely unknown. 
Using the \fbranch test, I found support for a modest level of introgression 
among several pairs of species which have been previously reported to hybridize. 
Most of which presently exist in sympatry with one another.
The highest \fbranch statistics were calculated between \amer and \hemiophrys and 
between \amer and \baxteri with values of 0.24 and 0.22 respectively \cref{fig:dsuite}.
One known \anaxyrus hybrid zones is one that exists between \amer and \hemiophrys.
\cite{green1983} reported clinal variation of allozyme alleles at five different 
loci across an approximately 100 km transect in southeastern Manitoba, Canada.
This sharp cline of variation suggests that reproductive isolation between these
species is quite high. 
It could be that introgression is occurring beyond this narrow hybrid zone
but the sample included in this study was sampled from a location in close 
proximity to the range of \amer \cref{fig:americanus-map} \parencite{conant1998}.
Thus, it is difficult to say if they detected introgression is shared by 
\hemiophrys as a whole or is only present within a hybrid zone.
Interestingly, there is also a high \fbranch score between \amer and \baxteri
which do no have ranges that are close to on another. 
It is possible that introgression from \amer occurred before the divergence  
of \hemiophrys and \baxteri. 
\baxteri is believed to be a relict of a more southerly distribution of 
\hemiophrys during a recent Pleistocene glacial period \parencite{henrich1968}.
Unfortunately, it is not possible to directly test for this scenario with \dsuite
due to limitations of the \fbranch test \parencite{malinsky2021}.

Several other \fbranch tests returned non-zero values although these were much
lower.
More than half of the \wood \fbranch statistics were greater than zero \amer \cref{fig:dsuite}.
Hybridization between all of these species is plausible as \wood occurs in 
sympatry at some part of its range with nearly all of them. 
Contemporary hybridization involving \wood has been reported with \amer, 
\cognatus, \microscaphus, and \speciosus \parencite{sullivan1986}. 
There is presently little to no overlap between \wood and \hemiophrys however
there could have been in the recent past as due to Pleistocene glaciation  
pushing the range of \hemiophrys further south \parencite{henrich1968}. 
The two non-zero \fbranch values for \quercicus with \punctatus and \speciosus 
are perplexing.  
The distribution of \quercicus is confined to the pine woodlands of the 
Southeastern United States whereas the other two species are found in the  
short arid grasslands and deserts of the Southwest \parencite{conant1998}. 
The \fbranch statistic for the comparison between \punctatus and the common 
ancestor of \speciosus and \cognatus is more plausible given their broadly
overlapping distributions in the present day \parencite{conant1998}.

Unfortunately, there were many comparisons among \anaxyrus that could not be 
made using the \fbranch test due to the structure of the phylogeny. 
Particularly among ancestral species between which past introgression could
be driving the pattern seen across the extant diversity which is a known 
caveat with the \fbranch test \parencite{malinsky2021}.
The results presented here are consistent with introgression being an 
important factor in the evolutionary history of \anaxyrus but gaps remain.
These are gaps that could potentially be addressed with more powerful 
likelihood or pseudo-likelihood methods. 

The d-statistic class of methods for detecting introgression are not able to   
test for introgression between sister species so could not shed any light 
on putative hybridization between \fowl and \wood. 
In order to test for introgression between \fowl and \wood I used the program 
structure along with PCA.  
The results of both \structure and PCA are consistent with the existence of  
a hybrid zone between these two species.
Two \wood samples, one from Arkansas and the other from Texas, have large proportions 
of inferred ancestry from \fowl. 
Several \fowl samples have large admixture proportions from \wood as well.  
The transition of ancestry proportions forms a steady East West gradient with 
one outlier present in Louisiana \cref{fig:structure-fowleri-woodhousii}.
The PCA results largely corroborate the results of the \structure analysis with 
\wood samples clustered tightly together, most \fowl samples clustering tightly
with a few deviating toward the center of the first principal compoenent axis, 
and finally two samples right in the center of the first principal component axis.
These results suggest the hybrid zone between \fowl and \wood is quite wide, on 
the order of hundreds of kilometers \cref{fig:structure-fowleri-woodhousii}.

This brings the number of confirmed \anaxyrus hybrid zones to three along with 
the \amer/\terr and \amer/\hemiophrys hybrid zones. 
Based on the \phycoeval phylogeny, all of these species emerged within the past 
2.5 million years.
This important context sheds light on the tempo of diversification within \anaxyrus.
The sister species pairs \fowl/\wood and \amer/\terr diverged only 0.7 and 1.0 mya
respectively \cref{fig:phycoeval}.
Neither of these species pairs has evolved a degree of reproductive isolation 
and/or character displacement that permits them to exist in sympatry with one 
another. 
Hybridization between species with older divergence times appears to be much 
less frequent with the exception of hybridization between \amer and \hemiophrys. 
\fowl occurs in sympatry across large regions with both \amer and \terr and 
\wood overlaps significantly with \amer \parencite{conant1998}.
This is likely possible due to a higher degree of reproductive isolation that 
has evolved between these species pairs in the form of differences in 
advertisement call and timing of reproduction ??Citation, possibly as a result 
of reinforcement.
This pattern suggests that pre-zygotic reproductive incompatibility tends to 
evolve more rapidly in \anaxyrus.
However, the hybrid zone between \amer and \hemiophrys complicates things 
as these two species are much more divergent from one another. 
Perhaps they have more recently come into secondary contact and there has not
been sufficient time to evolve pre-zygotic barriers to reproduction. 
This would support the idea that mating barriers between sympatric species 
have evolved through reinforcement and not under some other selective pressure. 


\subsection{Population Structure}
An examination of population structure can potentially provide clues about the 
historical factors that have shaped a species' diversification.
The geographic barriers that result in the reduction of gene flow within species
could be the same type of barrier that has resulted in past speciation events 
involving a species or its close relatives.
The \structure analysis of the \amer samples reveals population structure within 
this species which materializes as a gradient along three geographic axis at the center of  
which there is a high degree of admixture \cref{fig:structure-americanus}.
This variation corresponds loosely with the proposed subspecies 
\textit{A. anaxyrus charlesmithi} with a range that includes the Southwestern 
part of the \amer range in Oklahoma, Arkansas, and Missouri \parencite{bragg1954}.
This population structure, separates \amer into a Western population, a Southern population, and
a Northeastern population \cref{fig:structure-americanus}.
At the center of the range are highly admixed samples and with distance from 
this center, samples differentiate along the three different geographic axes.
This pattern is recapitulated in the PCA plots showing the same three axes of 
variation. 
Using the $delta$K method for choosing the best fitting \structure model  
favors the model with a K of two which separates \amer into Western and Eastern
populations. 
However, the K of three fits well with the results of the PCA, 
makes sense in a geographic context, and shows a more stark transition of 
admixture proportions from the Western group and Eastern groups.



The population \structure analyses conducted for \fowl, \terr, and \wood  
did not yield evidence for the existence of distinct populations within these 
species. 
Despite previous studies finding evidence for two distinct \wood populations 
\cite{masta2003,shannon1955}, I found \wood no evidence of this in my dataset.


% found two divergent clades of \wood using mitochondrial data.




\cite{masta2003} found two divergent clades of \wood using mitochondrial data.
\cite{conant1998} two distinct populations of \wood have been recognized on 
the basis of morphological differences with considering the southwest population
to be a subspecies \text{Anaxyrus woodhousii australis}
\cite{shannon1955} subspecies \textit{A. woodhousii australis} described on the  
basis of morphology.

It is possible that I simply did not include samples from the range of a 
distinct population.
I may not have sampled from this clade.



Differentiated samples of \wood may not have been included.


For \amer there is evidence for the existence of three somewhat differentiated groups 
that emerge along three different geographic axis at the center of which   
there is a high degree of admixture.



subspecies \textit{A. anaxyrus charlesmithi} \parencite{bragg1954}
subspecies \textit{A. woodhousii australis}


Population structure in A. woodhousii with two overlapping mtdDNA clades 
with one more associated with the Southwest and one more associated with the 
great planes \parencite{masta2003}





\subsection{Conclusion}
The evolutionary history of \anaxyrus inferred in this study has some important 
implications for understanding the contemporary hybridization in \anaxyrus


Given the great interest in hybridization within \anaxyrus and the promise of this 
group for furthering our understanding speciation, it is important to consider the 
implications of the evolutionary history inferred in this study.



