\section{Introduction}



% The degree of gene exchange that occurs in nature has been a highly debated topic. 
% Recent genomic studies have demonstrated that introgression is prevalent among 
% various groups of organisms, including fungi, vertebrates, insects, and angiosperms. 
% The consequences of introgression are diverse and depend on ecological and 
% genomic factors. 
% While it was previously believed to be mainly harmful, it is now evident that 
% introgression can be a source of genetic variation utilized in local adaptation 
% and adaptive radiation. Although our understanding of introgression as a 
% widespread phenomenon has improved, it remains uncertain how frequently it 
% occurs across different groups of organisms. 
% Ideally, determining the frequency of introgression throughout the tree of life 
% would involve analyzing clade-level genomic data systematically, without prior 
% knowledge of taxa known to hybridize in nature.



% Hybridization is a common phenomenon in nature and one which can have important 
% implications for evolution (Mallet, 2005). The role of hybridization in evolution 
% is becoming increasingly appreciated in a wide array of organisms and can have 
% varied effects on populations involved (A Genomic Perspective on Hybridization 
% and Speciation, 2016). Hybridization often results in reduced fecundity or the 
% production of unfit or sterile offspring when populations have sufficiently 
% diverged from one another. This cost has the potential to drive the evolution of 
% greater divergence between species when selection acts on traits that enhance 
% pre-mating barriers and reduce the likelihood hybridization (Servedio, 2001). 
% However, hybridization can also serve as a source of novel, adaptive genetic variation 
% through introduction of novel genetic variants or combinations thereof through 
% introgression between populations (Oziolor et al., 2019; Pardo-Diaz et al., 2012). 
% It has even been suggested that selection could maintain some degree of compatibility
% between species as a result of occasional introgression of adaptive genetic 
% variation (Servedio & Hermisson, 2020). In rare instances hybridization may 
% result in the emergence of new species from hybrid populations (Schumer et al., 2014). 
% If barriers to hybridization are not sufficiently strong, hybridization could 
% lead to the erosion of barriers between populations (Chafin et al., 2019). 
% Given that it can have important consequences in the evolution of lineages, 
% inference of historic introgression is important for a complete understanding of 
% a lineage’s evolutionary history and for understanding the consequences of 
% contemporary hybridization. 

% North American toads in the genus Anaxyrus have been the subject of many studies 
% of contemporary hybridization but we know very little about the historical context 
% preceding it. Many species pairs have been found to hybridize naturally or in the 
% laboratory (Blair, 1972; Green, 1996). Several attributes make toads appealing 
% for the study of hybridization. They are very abundant and conspicuous, and their 
% primary sexual signal – advertisement call— can be readily measured and quantified. 
% There has been success with rearing them in the lab and they produce very large clutch sizes. 
% Sex determination in toads is unique among species with historical prominence in
% hybridization research. Many bufonid toads are known to possess homomorphic sex chromosomes. 
% There have also been several transitions between XY and ZW sex determination. 
% This is interesting because many studies have found evidence for an important 
% role for sex chromosome in the evolution of reproductive incompatibility (Coyne & Orr, 2004). 
% But most studies have focused on species with heteromorphic sex chromosomes 
% where hybridization occurs between species with XY sex determination systems.

% While North American toads are an attractive system for gaining more understanding 
% of the process of speciation and evolution of reproductive incompatibility, 
% we currently lack important historical context for hybridization in these toads. 
% Foremost is uncertainty about the evolutionary relationships among species and populations. 
% It is not clear how closely related hybridizing species are to one another and whether 
% they are each other’s closest living relatives. A number of studies have estimated their 
% phylogenetic relationships, but there is some disagreement among these studies 
% (Fontenot et al., 2011; Graybeal, 1997; Masta et al., 2002; Pramuk et al., 2008; Pyron et al., 2011). 
% Particularly in the americanus group which is composed of 7 currently recognized species. 
% The inconsistencies in inferred relationships could be due to many possible 
% sources of error known to affect phylogenetic inference (Som, 2014). 
% Furthermore, some studies only included a single sample from currently 
% recognized species, and thus were not able to confirm the monophyly of these species 
% (Pyron & Wiens, 2011). In studies that did sample multiple individuals per
% species, some, like A. americanus and A. fowleri, were inferred to be polyphyletic (Fontenot et al., 2011).
% The inconsistencies in inferred relationships among Anaxyrus species could be caused by 
% real biological processes such as incomplete lineage sorting or introgressive 
% hybridization (Degnan & Rosenberg, 2009; Kubatko & Degnan, 2007).
% Incomplete lineage sorting or hybridization could result in individual gene 
% trees that differ from the true species history. Graybeal, 1997 and Masta et al., 2002 used only mitochondrial sequence data which may not be consistent with the evolutionary history of the nuclear genome or that of the species due to incomplete lineage sorting or mitochondrial introgression. Fontenot et al., 2011, Pramuk et al., 2008, and Pyron & Wiens, 2011 used concatenated alignments of mtDNA and a small number of nuclear loci. However, concatenated analyses can be misleading when there is high discordance among gene trees due to incomplete lineage sorting (Kubatko & Degnan, 2007; Roch & Steel, 2015). Given the large number of putative hybrids observed between contemporary populations of toads, historic hybridization resulting in a significant amount of gene tree discordance seems highly plausible. Quantifying how much discordance is caused by hybridization would also be important context for understanding contemporary hybridization and the evolution of reproductive barriers between species. A history of hybridization and admixture among ancestral lineages could play an important role in the degree of reproductive isolation that exists between species of toads today. 
% Knowing when species have diverged is also important context for understanding 
% contemporary hybridization and the evolution of reproductive isolation. 
% It is still unknown how time since divergence factors into the evolution of 
% reproductive incompatibility. Knowledge of divergence times can also shed light 
% on drivers of speciation. For example, divergence times that are shared among 
% different pairs of species are particularly interesting in that they suggest the 
% processes that have driven diversification may have been shared among taxa.

% I propose to obtain genome wide sequence data from samples representing a 
% large portion of the ranges of most described species in the genus Anaxyrus. 
% With these data, I will test the following hypotheses: 1) recent biogeographic 
% processes have reduced or eliminated gene flow between populations within species; 
% 2) contemporary hybridization occurs between species with overlapping or abutting ranges; 
% 3) ancient introgression has occurred between lineages; 
% 4) large-scale biogeographic processes have driven diversification in Anaxyrus; 
% and 5) the current taxonomy of Anaxyrus is not consistent with evolutionary 
% relationships among populations within named groups. 





\section{Methods}
\subsection{Sampling and DNA Isolation}
I obtained tissue samples museum tissue collections as well as from individuals   
that I collected from 2017 to 2020. I selected samples to represent as much
of the range of each species of \textit{Anaxyrus} as possible.
I also included one \textit{Rhinella marina} and one \textit{Incillius nebulifer} 
for use as outgroups for phylogenetic analyses.

I isolated DNA from tissues by first lysing a piece of tissue approximately 
the size of a grain of rice in 300 \uL\ of a solution of 10mM Tris-HCL, 10mM EDTA, 
1\% SDS (w/v), and nuclease free water along with 6 mg Proteinase K that was 
incubated for 4-16 hours at 55\degree C in a 1.5 mL microcentrifuge tube.  
To purify the DNA and separate it from the lysis product, I mixed the lysis 
product with a 2X volume of SPRI bead solution containing 1 mM EDTA,  
10 mM Tris-HCl, 1 M NaCl, 0.275\% Tween-20 (v/v), 18\% PEG 8000 (w/v), 
2\% Sera-Mag SpeedBeads (GE Healthcare PN 65152105050250) (v/v), and nuclease free water.
I then incubated the samples at room temperature for 5 minutes, placed the 
beads on a magnetic rack, and discarded the supernatant once the beads had collected
on the side of the tube.  
I then performed two ethanol washes by adding 1 mL of 70\% ETOH to the beads
while still placed in the magnet stand and allowing it to stand for 5 minutes
before discarding the ethanol. 
After removing all ethanol from the second wash, I removed the tube from the magnet 
stand and allowed the sample to dry for 1 minute before mixing the beads with 100 \uL\ of 
TLE solution containing 10 mM Tris-HCL, 0.1 mm EDTA, and nuclease free water.
After allowing the bead mixture to stand at room temperature for 5 minutes I returned
the beads to the magnet stand, pipetted all of the TLE solution into another 
microcentrifuge tube, and discarded the beads. I quantified DNA with a Qubit
fluorometer (Life Technologies, USA) and diluted samples with TLE solution to 
bring the concentration to 20 ng/\uL.

\subsection{RADseq Library Preparation}
I prepared RADseq libraries using the 2RAD approach outlined by \cite{bayona-vasquez2019}. 
On 96 well plates, I ligated 100 ng of sample DNA in 15 \uL\ of a solution with 
1X CutSmart Buffer (New England Biolabs, USA; NEB), 10 units of XbaI,
10 units of EcoRI, 0.33 \uM\ XbaI compatible adapter, 0.33 \uM\ EcoRI compatible adapter,
and nuclease free water with a 1 hour incubation at 37\degree C. 
I then immediately added 5 \uL\ of a solution with 1X Ligase Buffer (NEB),
0.75 mM ATP (NEB), 100 units DNA Ligase (NEB), and nuclease free water 
and incubated at 22\degree C for 20 min and 37\degree C for 10 min for two cycles, 
followed by 80\degree C for 20 min to stop enzyme activity.
For each 96 well plate, I pooled 10 \uL\ of each sample and split this pool 
equally between two microcentrifuge tubes.
I purified each pool of libraries with a 1X volume of SpeedBead solution followed 
by two ethanol washes as described in the previous section except that the DNA 
was resuspended in 25 \uL\ of TLE solution. 

In order to be able to detect and remove PCR duplicates, I performed a single   
cycle of PCR with the iTru5-8N primer which adds a random 8 nucleotide barcode to 
each library construct.  
For each plate, I prepared four PCR reactions with a total volume of 
50 \uL\ containing 1X Kapa Hifi Buffer (Kapa Biosystems, USA; Kapa),
0.3 \uM\ iTru5-8N Primer, 0.3 mM dNTP, 1 unit Kapa HiFi DNA Polymerase,
10 \uL\ of purified ligation product, and nuclease free water.
I ran reactions through a single cycle of PCR on a thermocycler at 98\degree C for 2 min, 
60\degree C for 30 s, and 72\degree C for 5 min. 
I pooled all of the PCR products for a plate into a single tube and purified the
libraries with a 2X volume of SpeedBead solution as described before and 
resuspended in 25 \uL\ TLE.
I added the remaining adapter and index sequences unique to each plate with four PCR
reactions with a total volume of 50 \uL\ containing 1X Kapa Hifi (Kapa),
0.3 \uM\ iTru7 Primer, 0.3 \uM\ P5 Primer, 0.3 mM dNTP, 1 unit of Kapa Hifi DNA Polymerase (Kapa),
10 \uL\ purified iTru5-8N PCR product, and nuclease free water.
I ran reactions on a thermocycler with an initial denaturation at 98\degree C for 2 min, 
followed by 6 cycles of 98\degree C for 20 s, 60\degree C for 15 s, 72\degree C 
for 30 s and a final extension of 72\degree C for 5 min.
I pooled all of the PCR products for a plate into a single tube and purified the
product with a 2X volume of SpeedBead solution as described before and 
resuspended in 45 \uL\ TLE.

I size selected the library DNA from each plate in the range of 450-650 base pairs using
a BluePippin (Sage Science, USA) with a 1.5\% dye free gel with internal R2 standards. 
To increase the final DNA concentrations I prepared four PCR reactions for each 
plate with 1X Kapa Hifi (Kapa), 0.3 \uM\ P5 Primer, 0.3 \uM\ P7 Primer, 0.3 mM dNTP, 
1 unit of Kapa HiFi DNA Polymerase (Kapa), 10 \uL\ size selected DNA, and 
nuclease free water and used the same thermocycling conditions as the previous
(P5-iTru7) amplification.
I pooled all of the PCR products for a plate into a single tube and purified 
the product with a 2X volume of SpeedBead solution as before and resuspended in 20 \uL\ TLE. 
I quantified the DNA concentration for each plate with a Qubit fluorometer 
(Life Technologies, USA) then pooled each plate in equimolar amounts relative 
to the number of samples on the plate and diluted the pooled DNA to 5 nM with
TLE solution. 
The pooled libraries were pooled with other projects and sequenced on an Illumina 
HiSeqX by Novogene (China) to obtain paired end, 150 base pair sequences. 

\subsection{Data Processing}

I demultiplexed the iTru7 indexes using the \processradtags command from 
Stacks2 v2.6.4 \parencites{rochette2019} and allowed for two mismatches for rescuing reads.
To remove PCR duplicates, I used the \clonefilter command from \stacks.
I demultiplexed inline sample barcodes, trimmed adapter sequence, and filtered 
reads with low quality scores as well as reads with any uncalled bases using the  
\processradtags command again and allowed for the rescue of restriction site sequence 
as well as barcodes with up to two mismatches.  



% I demultiplexed plate indexes using the \processradtags command from 
% Stacks2 v2.6.1 \parencite{rochette2019} and allowed for 2 mismatches for rescuing reads. 
% I demultiplexed samples and trimmed, filtered, and assembled reads with 
% ipyrad v0.9.9 \parencite{eaton2020}. I specified the datatype as "pair3rad", 
% used a clustering threshold of 80\% sequence similarity, and allowed for two  
% mismatches in the barcode sequence. The remaining parameters were left with the 
% default settings.
% I used Ipyrad to filter loci that were absent in greater than 50\% of samples and then filtered
% samples with fewer than 200 loci. 



% \subsection{Phylogenetics}

% Excluded any samples with fewer than 500 loci for phylogeny estimation with ipyrad filtering


% Check for evidence of hybrids using 
% program STRUCTURE v2.3.4 \parencite{pritchard2000}
% PCA using adagenet v2.1.8 \parencite{jombart2011}
% Exclude any?

\subsection{Maximum Likelihood}

Maximum likelihood using IQ-TREE v1.6.12 \parencite{nguyen2015} 
with 1000 ultrafast bootstrap replicates \parencite{hoang2018}
under GTR substitution model.

\subsection{Structure}

Run for 100,000 iterations with 50,000 burnin


Run structure with all americanus group to make sure there aren't hybrids.
Run structure with american toad, southern toad, woodhousii, and fowleri to see if there is population structure 

Filtered loci not found in 75%
Filtered variants with minor allele count less than 3.
Filtered sites with more than 2 alleles














\subsection{Phycoeval}
To see if there is evidence of shared divergence times and to estimate a species tree under multispecies coalescent
Phycoeval v1.0.0 \parencite{oaks2022} 

\subsection{Dsuite}
To look for evidence of past admixture.
% I randomly sampled a single SNP from each locus from each locus using a  
% custom python script relying on the scikit-allel v1.3.5 package.
% Using vcftools v0.1.17 \parencite{danecek2011}, I dropped any individuals that 
% had more than 25\% missing sites and filtered individuals that had more than 
% 35\% missing data after the missing sites filter was applied. 
Dsuite \parencite{malinsky2021}


\section{Results}
Average number of reads per individual
Mean coverage per locus

Total loci and snps after filtering

\subsection{Maximum Likelihood}
Inferred a single well supported clade (>***\%) for each recognized species. 




\section{Discussion}


Population structure in A. woodhousii with two overlapping mtdDNA clades 
with one more associated with the Southwest and one more associated with the 
great planes \parencite{masta2003}

Phylogeny of toads, doesn't place Americanus and Terrestris as sister, finds
mitonuclear discordance and Finds two fowleri clades \parencite{fontenot2011}

Inconsistent phylogeny \parencite{masta2002}
Inconsistent phylogeny \parencite{pramuk2007}
Inconsistent phylogeny \parencite{graybeal1997}
Inconsistent phylogeny \parencite{pyron2011}

Idea:
Given the appearance that there are many secondary contact zones. It seems 
probable that toad species have undergone range expansions. Following these
range expansions, are there any barriers that are now reducing gene flow?
We can test that by looking for population structure within species that 
aligns with possible biogeographic barriers.