\newcommand{\figlegend}{The left column shows estimates from \beast,
and the center and right column shows estimates from \ecoevolity
using all sites and (at most) one SNP per locus.
The top row shows estimates from 200 data sets simulated without
character-pattern errors.
Rows labelled 20\% and 40\% singleton errors show estimates from the same
alignments after singleton site patterns were changed to invariant sites with
probabilities 0.2 and 0.4, respectively.
Rows labelled 20\% and 40\% het errors show estimates from the same
(error-free) alignments after we randomly paired gene copies within each
species into two diploid genotypes, and for each genotype we randomly replaced
one allele with the other with probability 0.2 and 0.4, respectively.
Each plotted circle and associated error bars represent the posterior mean and 95\%
credible interval.
Circles and error bars are colored yellow if the effective sample size (ESS) of
the estimate was less than 200, red if the potential scale reduction factor
(PSRF) was greater than 1.2, and green if both conditions were true.
The root mean square error (RMSE) and rate of poor MCMC behavior (RPMB) is
given for each plot, the latter of which is the proportion of estimates with
ESS < 200 and/or PSRF > 1.2.
We generated the plots using matplotlib Version 3.1.1 \parencite{matplotlib}.}
