\section{Introduction}
% A complete understanding of the processes that lead to Haldane’s rule (HR)—the 
% near ubiquitous tendency for the sex with non-identical sex chromosomes (heterogametic sex) 
% to suffer more from reproductive incompatibility in the form of sterility or 
% inviability following hybridization—has yet to be fully realized. 
% An understanding of such a rigidly held phenomenon will be key for gaining a more 
% complete understanding of the processes that drive the origin of species which is 
% one of the most enduring pursuits in biology. There are several non-exclusive 
% processes hypothesized to explain HR which are supported by both theory and 
% empirical data. It is expected that there is interplay among these processes, 
% but some may not be relevant to all groups of organisms (Delph and Demuth, 2016). 
% The “dominance hypothesis” for example, which is the most prominent hypothesis 
% explaining HR is only applicable to organisms with differentiated sex chromosomes 
% (heteromorphic) (Delph and Demuth, 2016). Under the dominance hypothesis, 
% reproductive incompatibility is caused by an interaction between a recessive locus 
% on the sex chromosome that is possessed by both sexes (X or Z chromosome) and 
% another locus on a different chromosome (Turelli and Orr, 1995). This incompatibility 
% is only expressed in the heterogametic sex due to the presence of a single copy 
% of the locus and therefore, is unlikely to explain HR in species with identical 
% sex chromosomes (homomorphic) as there are two copies of every or nearly every 
% locus in both sexes (Turelli and Orr, 1995). Another leading hypothesis is the 
% “faster-male hypothesis” which posits that genes related to the reproductive 
% systems of males undergo more rapid evolution or are more susceptible to 
% disruption in hybrids (Delph and Demuth, 2016). The “faster-male hypothesis” 
% cannot explain HR in organisms in which females are the heterogametic sex (ZW). 
% Neither of these hypotheses predict that HR would apply to species with homomorphic
% ZW sex chromosomes, yet greater inviability and sterility in females (the heterogametic sex) 
% has been found in such species (Malone and Fontenot, 2008). None of these species 
% have been studied in detail to examine the contributions and nature of other 
% hypothesized processes behind Haldane’s rule in the absence of others that have 
% received more attention. One way that HR can test for is by taking advantage of
% hybrid zones which provide a useful opportunity to observe the interaction of 
% divergent gene pools allowing for the investigation of the underlying processes
% that cause reproductive incompatibility (Gompert et al., 2017).


% Hybrid zones are geographic areas where divergent lineages interbreed and can 
% lead to the exchange of genetic variation between the involved lineages (i.e. introgression). 
% This genetic exchange can serve as a source of adaptive genetic variation in 
% recipient lineages (Elgvin et al., 2017). But hybridization can be costly when 
% reproductive incompatibility results in the production sterile or unfit offspring 
% (Barton and Hewitt, 1985). In these cases, selection might favor individuals that 
% mate assortatively and do not produce unfit offspring which would further increase 
% divergence between the two lineages—particularly in traits that enhance prezygotic 
% isolation (Servedio, 2000).

% A powerful tool for studying the interaction of divergent genomes at hybrid zones 
% is the cline model which quantifies genetic variation across hybrid zones. There 
% are two classifications of cline models: (1) geographic clines and (2) genomic 
% clines. Geographic cline models quantify variation across a linear transect of 
% a hybrid zone (Barton and Hewitt, 1985). These models make the assumption that 
% allele frequencies change monotonically from one side of the zone to the other, 
% but this assumption is sometimes violated such as in the case of mosaic hybrid 
% zones (Harrison, 1986). Genomic clines are not spatially explicit and quantify 
% genetic variation along an admixture gradient (Barton and Gale, 1993). An 
% admixture gradient is the varying proportion of genomes from two hybridizing 
% lineages present in the recombinant offspring of multi-generation hybrids. 
% Genomic clines can be utilized in cases where there is no obvious spatial axis 
% (Gompert et al., 2017).

% Cline analysis can quantify locus-specific introgression relative to the genome 
% wide average. Outlier loci are regarded as putatively under strong selection for 
% or against introgression based on relative position to the genome wide average. 
% These estimates can provide insight into the number of loci that are involved in 
% reproductive incompatibility, the relative strength of selection among loci, as 
% well as the overall strength of incompatibility between hybridizing lineages 
% (Gompert et al., 2017). Cline analysis has also been used to identify putatively 
% adaptive loci that introgress more readily between hybridizing lineages (Chhatre et al., 2018). 
% It can also be used detect asymmetrical gene flow indicative of sex biased 
% dispersal or asymmetric effects of reproductive incompatibility or fitness 
% (Barton and Hewitt, 1985). Estimates of differential rates of introgression 
% are also useful for understanding the genomic architecture of incompatibility 
% by revealing the location of loci associated with reproductive incompatibility (Janoušek et al., 2015).

% Cline analysis can also be used to determine if the outcome of hybridization 
% between two species is consistent with the expectations of HR. The rate of 
% introgression of any loci uniparentally inherited from the heterogametic sex
% are expected to be reduced as the individuals from which these loci are inherited 
% will be affected by reduced viability and fertility when the hybridizing species 
% are affected by HR. This would affect the rate of introgression of loci such as 
% those on Y chromosomes, W chromosomes, mitochondrial genomes, or the chloroplasts 
% of many plants. Clines of uniparentally inherited loci would be the steeper and 
% narrower than the clines of loci from other parts of the genome.

% Toads in the family Bufonidae have homomorphic sex chromosomes and most have 
% been shown to have ZW sex determination (Brelsford et al., 2013). Laboratory 
% crosses involving 92 different species indicate that most species are affected 
% by HR but that some could be rare exceptions (Malone and Fontenot, 2008). A 
% number of hybrid zones are known or suspected to exist between toad species 
% (Green 1996). One hybrid zone suspected on the basis of morphological data 
% involves the American toad and Southern toad in central Alabama, (Weatherby, Craig A, 1982). 
% The ranges of these two species do not overlap but do meet at a long zone of 
% contact in the Southeastern United States corresponding with a prominent 
% geological transition known as the “fall line” (Shankman and Hart, 2010). 
% The fall line separates the Appalachian highlands to the North from the coastal 
% plain to the South (Shankman and Hart, 2010). This pattern of non-overlapping 
% distribution is consistent with the existence of a tension zone between the two 
% species (Key, 1968). Tension zones form where an equilibrium is reached between 
% selection and migration resulting in the maintenance of species boundaries despite 
% ongoing hybridization (Barton and Hewitt, 1985). Tension zones are expected to 
% correspond with a transition in ecological conditions and with areas where both 
% species occur at lower abundance (Barton and Hewitt, 1985). Laboratory crosses 
% between these two species have been shown to result in viable and fertile first 
% generation male and female hybrid offspring (Blair, 1963). Second generation 
% offspring that result from back crossing first generation offspring to the parent 
% species do suffer from a partial reduction in viability during development, but 
% it has not been determined if one sex suffers more from this or if either sex 
% suffers from infertility (Blair, 1963). If hybrid offspring fertility and viability 
% is not consistent with HR, it would make this one of the exceptionally rare examples 
% known making it an excellent target for comparative study to understand the mechanisms 
% that generate HR.

% Below I propose a study using genome wide data to investigate the consequences 
% of hybridization of American and Southern toads in central Alabama and assess 
% the applicability of HR in the hybrids of these two species. I will 1) test the 
% hypothesis that introgression occurs between American and Southern toads where 
% the ranges of these two species meet in central Alabama and 2) test the hypothesis 
% that hybrid females have reduced viability or sterility consistent with Haldane’s Rule.

% %%%%%%%%%%%%%%%%%%%%%%%%%%%
% % New stuff

% Effects of hybridization
Speciation is the process by which genetic divergence leads to reproductive  
isolation between divergent lineages.
It is a continuous process during which there may be ongoing gene flow or 
introgression via hybridization following a period of isolation and subsequent
secondary contact \parencite{mallet2008,wu2001}.
Introgression is possible because genetic barriers to introgression that accumulate 
within the genome are a property of genomic regions rather than a property of the 
entirety of the genome \parencite{wu2001,gompert2012b}.
Natural hybridization between divergent lineages has become increasingly 
appreciated as a widespread phenomenon in recent years \parencite{mallet2005,moran2021}.
It is a phenomenon that can have important evolutionary consequences.
Hybridization can be a source of adaptive variation \parencite{hedrick2013}. 
It can also introduce deleterious genetic load which persists long term 
within a population \parencite{moran2021}. % and can create more boundaries? 
Hybridization can create conditions where selection favors the evolution of traits 
that enhance assortative mating and reduce the production of unfit hybrid offspring
which drives further genetic divergence and reinforcement of reproductive barriers
between lineages \parencite{servedio2003}.
If hybrids do not suffer any negative fitness effects, hybridization could lead to 
the erosion of differences between divergent populations \parencite{taylor2006}.
Potentially resulting in populations that are genetically distinct from either 
parent species which can themselves eventually evolve reproductive isolation from 
the parent species \parencite{moran2021}.

% Uses of hybridization
Aside from having important evolutionary consequences which need to be understood, 
hybridization is also an excellent opportunity to investigate the processes
that result in the evolution of reproductive incompatibility and divergence between 
evolutionary lineages. 
Hybrid zones are particularly suitable due to the production of a large numbers of 
recombinant genomes carrying many possible combinations of genomic elements from parent 
species resulting from many generations of backcrossing \parencite{rieseberg1999}. 
The many generations of backcrossing and recombination make it possible to 
distinguish between the effects of closely linked genes \parencite{rieseberg1999}. 
The many generations and large number of individuals producing these genetic 
combinations are not feasible to produce experimentally in the vast majority of
species \parencite{rieseberg1999}. 
Furthermore, the combination of genes produced are exposed to selection 
under natural conditions. 
This is important as the effect of hybrid incompatibilities can be dependent on 
on environmental conditions and can only truly understood in this context \parencite{miller2016}. 

% Standing questions
Despite being a fundamental evolutionary process, our understanding of speciation 
is far from complete \parencite{butlin2011}. 
Only a few loci, in a few species, have been pinpointed as the direct cause of 
reproductive incompatibility between species \parencite{blackman2016,nosil2011}.
Consequently, our understanding of the processes that drive the evolution of loci
resulting in reproductive incompatibility is limited \parencite{butlin2011}. 
Studies of introgression within hybrid zones have identified highly variable
rates of introgression among loci \parencite{barton1985,gompert2017}.
This heterogeneity can arise via genetic drift occurring within hybrid zones,
but will also be caused by differences among loci in the strength of selection 
against them in a hybrid genomic background \parencite{barton1985,gompert2017}. 
It has also been observed that the levels of genetic divergence between species 
are highly variable across the genome \parencite{nosil2009}.
Much of this heterogeneity is the result of divergent selection acting on each 
species independently \parencite{nosil2009}.
Regions with particularly high levels of divergence between closely related species have been 
coined "genomic islands of divergence" \parencite{wolf2017}. 
It is assumed, particularly in the case of speciation with gene flow, that these
genomic island harbor genes that reduce interbreeding between species. 
When speciation occurs with gene flow, divergent selection can cause 
adaptive divergence in habitat use, phenology, or mating signals, and reduce the
frequency or success of interspecific matings. 
When species diverge in geographic isolation, divergent selection and reproductive
isolation could be decoupled and reproductive isolation could just be a result
of genetic drift.
Whether loci under divergent selection between two species also contribute to 
reproductive isolation has not been widely explored.
A handful of studies have found evidence for a modest relationship between 
genetic divergence and selection against introgression \parencite{nikolakis2022,gompert2012a,parchman2013,larson2013}.
How consistent and widespread this pattern is remains to be seen.
At least one study has found no association \parencite{jahner2021}.

% Although there is a substantial body of theoretical work, there is a limited 
% amount of empirical data to rigorously test it. 
% The Dobzhansky-Muller model is widely accepted as the mode by which incompatibilities
% can evolve \parencite{bateson1909,dobzhansky1936,muller1942}.
% However, this model is very general and only predicts that reproductive incompatibility 
% results from epistatic interactions between two or more loci.
% More specific predictions are that they arise because of...
% List hypothesized forces that drive evolution of reproductive incompatibility... 
% Structural variation, 
% One interesting and unanswered question is whether or not there is a strong 
% relationship between divergence and reproductive incompatibility.
% There has been strong interest in identifying regions of elevated genetic 
% divergence between the genomes of closely related species. 
% The assumption being that these regions have been resistant to the homogenizing
% effect of hybridization or have undergone intense selection on traits that 
% Researchers have been interested in identifying highly divergent portions of genomes
% naming them so called "Islands of speciation".
% However few have tested for an association between these divergent regions
% and incompatibility. (Except see \cite{gompert2012a}, Larset et al. 2013, and Parchman et al 2013, cited in Gompert )
% During speciation with gene flow, divergent selection cause adaptive 
% divergence in traits such as habitat, phenology, or mating signal \parencite{gompert2012a} 
% When speciation occurs without gene flow, incompatibility loci may arise via  
% drift rather than selection.
% Unanswered questions:
% Number of loci
% Distribution of loci
% Strength of selection
% Type of selection
% - local adaptation
% - sexual selection
% - host-pathogen
% What is the roll of drift?
% Traits under selection
% What conditions promote speciation
% Role of plasticity
% Gene expression

% The patterns that are studied and how:
% The patterns introgression across a hybrid zone reveal the important differences
% Which loci act as barriers to successful reproduction.
% Can measure the relative extent of introgression through a hybrid zone to
% identify loci that are selected against when they occur in hybrid genetic background
% Studies have found there to be heterogeneity in the degree to which loci can introgress.
% Clines are a tool to quantify this.
% Can tell us how and why genetic differences arise
% How many genetic changes are necessary?
% Do they arise via selection or drift?
% Is it related to genetic differentiation?
% What we know about how incompatibilities arise
% DMIs \parencite{bateson1909,dobzhansky1936,muller1942}
% It has also become apparent that there is a significant degree of heterogeneity 
% across the genome in the extent of both genetic differentiation and introgression 
% between divergent lineages \parencite{wu2001,nosil2009}.
% This heterogeneity arises from the varied affects that loci have on fitness
% in parent populations and in individuals that have hybrid ancestry \parencite{nosil2009}\citationNeeded.
% We have numerous examples to what causes heterogeneity in divergence \citaionNeeded.
% Examples are \ldots. 
% Precisely how and why genetic differences which reduce the fitness of hybrids arise 
% is still not well understood. 
% Models of how it can occur have existed for some time but we have limited 
% empirical evidence.
% Dobzhansky Muller model
% They can be intrinsic and reduce viability or fertility.
% They can extrinsic and be due to interaction with environmental conditions
% What is the link between divergence and incompatibility?
% Rapidly diverging parts of the genome would seem to be obvious candidates 
% for harboring incompatibility factors.
% Hybird zones are stable over long periods of time \parencite{deraad2023}
% Is important to for understanding how divergent lineages evolve  
% and persist when they exist sympatry either because of secondary contact or
% sympatric speciation.

% Southern toad american toad hybrid zone
In this study I investigate hybridization between the American toad 
(\textit{Anaxayrus americanus})and Southern toad (\textit{Anaxyrus terrestris}) 
at a suspected hybrid zone in the Southern United States to assess the extent of 
introgression between them and test for a relationship between introgression and 
genetic divergence. 
This suspected hybrid zone has not been investigated with genetic data previously 
but it bears many hallmarks of a tension zone \parencite{barton1985}. 
Under the tension zone model of hybridization, species boundaries are maintained
by a balance between dispersal and selection against individuals carrying incompatible 
hybrid genotypes \parencite{barton1985}. 
The ranges of \amer and \terr abut with an abrupt transition and no apparent overlap
along a long contact zone which from Louisiana to Virginia. 
This contact zone closely corresponds with a prominent physiographic feature known 
as the "fall line" \parencite{powell2016,mount1975}. 
The Fall line is the boundary between the Southern coastal plain to the South and 
the Appalachian Highlands to the North \parencite{shankman2007}.
These regions differ in their underlying geology, topography, and elevation \parencite{shankman2007}.
The distribution of \terr is restricted to the coastal plain extending from   
the Mississippi River in the West to Virginia in the East (\cref{fig:hybrid-main}).
The distribution of the American Toad encompasses nearly all of the Eastern North 
American with the exception of the Southern coastal plain (\cref{fig:hybrid-main}).
Tension zones are expected to correspond with natural features that reduce
dispersal or abundance \parencite{barton1979}.
Such a sudden transition is difficult to explain if not the result of 
the processes characteristic of tension zones. 
For there to be no mutually hospitable areas permitting some range overlap is 
implausible without there being an extreme level of competition or extreme degree 
of adaptation by each species to their respective environments.
The two species have only slight differences in male advertisement call and in 
morphological appearance \parencite{cocroft1995,weatherby1982}. 
They only differ slightly in the timing of their spawn \parencite{mount1975} 
However, there is some overlap in the spawning period and male Bufonidae 
are famously indiscriminate in their choice of mates \parencite{dordevic2014,weatherby1982}.
They have also been shown to have a degree of reproductive compatibility through 
laboratory crossing experiments which produced viable $F_2$ offspring \parencite{blair1963}. 
Analysis of morphological variation in central Alabama by \cite{weatherby1982} 
suggests there has been introgression between them. 

% History and prevalence of natural hybridization in toads 
The "true toads" in the family Bufonidae, to which \amer and \terr belong,  
have been a prominent group of organisms in the literature on hybridization. 
W.F. Blair and colleagues performed a remarkable 1,934 separate experimental crosses 
to quantify the degree of reproductive incompatibility between species pairs within
this family \parencite{blair1972,malone2008}.
These experiments demonstrated a high degree of compatibility between some closely 
related species pairs in which hybrids were capable of producing viable backcross 
or $F_2$ hybrid offspring \parencite{blair1963}.
Furthermore, numerous cases of natural hybridization among toad species have been 
reported with several apparent or clear hybrid zones 
\parencite{green1996,vanriemsdijk2023,colliard2010,weatherby1982}.
Despite the interest in and appreciation for hybridization in Bufonidae, only a 
small amount of work has been done to understand patterns of introgression
within hybrid zones. 
A clinal pattern of admixture at 26 allozyme loci has been shown within the 
\textit{Anaxyrus americnaus} X \textit{Anaxyrus hemiophrys} hybrid zone in 
Ontario, Canada\parencite{green1983}.
Almost no admixture was detected at 7 microsatellite loci within the suspected \textit{Bufo siculus} X 
\textit{Bufo balearicus} hybrid zone in Sicily, Italy \parencite{colliard2010}.
The most comprehensive study of introgression within a Bufonidae hybrid zone found 
significant levels of genome wide admixture, fitting a clinal pattern, at two separate transects 
at either end of the \textit{Bufo bufo} x \textit{Bufo spinosus} hybrid zone
in Southern France \parencite{vanriemsdijk2023}.

% Synopsis and major questions
The suspected \amer, \terr hybrid zone has great potential to expand our 
understanding of speciation. 
This will be dependent on the degree of ongoing introgression, if any, between 
these species.
In this study I use genome-wide sequence data to characterize patterns of  
introgression within the hybrid zone using model based inference of admixture
proportions, bayesian genomic cline analysis, and estimates of parental population differentiation.
With these approaches I specifically address the following questions: 
1) Is there evidence of ongoing hybridization and admixture between the two species,
2) Do any loci have outstanding patterns of introgression consistent with them
being linked to reproductive incompatibility, and
3) Is there any relationship between patterns of introgression and levels 
of genetic differentiation between parental lineages?


\section{Methods}
\subsection{Sampling and DNA Isolation}
I collected genetic samples from \textit{A. americanus} and \textit{A. terrestris}
by driving roads during rainy nights between 2017 and 2020
in an a region of central Alabama where hybridization has previously been
inferred from the presence of morphological intermediates \parencite{weatherby1982}. 
I euthanized individuals with immersion in buffered MS-222.
I removed liver and/or toes and preserved them in 100\% ethanol and fixed 
specimens with 10\% Formalin solution.
Genetic samples and formalin fixed specimens were deposited at the Auburn Museum of Natural History.
Additional samples were also provided by museums (see \cref{tab:loanedHyb}).

I isolated DNA by lysing a small piece of liver or toe approximately 
the size of a grain of rice in 300 \uL\ of a solution of 10mM Tris-HCL, 10mM EDTA, 
1\% SDS (w/v), and nuclease free water along with 6 mg Proteinase K and 
incubating for 4-16 hours at 55\degree C.  
To purify the DNA and separate it from the lysis product, I mixed the lysis 
product with a 2X volume of SPRI bead solution containing 1 mM EDTA,  
10 mM Tris-HCl, 1 M NaCl, 0.275\% Tween-20 (v/v), 18\% PEG 8000 (w/v), 
2\% Sera-Mag SpeedBeads (GE Healthcare PN 65152105050250) (v/v), and nuclease free water.
I then incubated the samples at room temperature for 5 minutes, placed the 
beads on a magnetic rack, and discarded the supernatant once the beads had collected
on the side of the tube.  
I then performed two ethanol washes by adding 1 mL of 70\% ETOH to the beads
while still placed in the magnet stand and allowing it to stand for 5 minutes
before removing and discarding the ethanol. 
After removing all ethanol from the second wash, I removed the tube from the magnet 
stand and allowed the sample to dry for 1 minute before thoroughly mixing the beads with 100 \uL\ of 
TLE solution containing 10 mM Tris-HCL, 0.1 mm EDTA, and nuclease free water.
After allowing the bead mixture to stand at room temperature for 5 minutes I returned
the beads to the magnet stand, collected the TLE solution, and discarded the beads. 
I quantified DNA in the TLE solution with a Qubit
fluorometer (Life Technologies, USA) and diluted samples with additional TLE solution to 
bring the concentration to 20 ng/\uL.

\subsection{RADseq Library Preparation}
I prepared RADseq libraries using the 2RAD approach developed by \cite{bayona-vasquez2019}. 
On 96 well plates, I ligated 100 ng of sample DNA in 15 \uL\ of a solution with 
1X CutSmart Buffer (New England Biolabs, USA; NEB), 10 units of XbaI,
10 units of EcoRI, 0.33 \uM\ XbaI compatible adapter, 0.33 \uM\ EcoRI compatible adapter,
and nuclease free water with a 1 hour incubation at 37\degree C. 
I then immediately added 5 \uL\ of a solution with 1X Ligase Buffer (NEB),
0.75 mM ATP (NEB), 100 units DNA Ligase (NEB), and nuclease free water 
and incubated at 22\degree C for 20 min and 37\degree C for 10 min for two cycles, 
followed by 80\degree C for 20 min to stop enzyme activity.
For each 96 well plate, I pooled 10 \uL\ of each sample and split this pool 
equally between two microcentrifuge tubes.
I purified each pool of libraries with a 1X volume of SpeedBead solution followed 
by two ethanol washes as described in the previous section except that the DNA 
was resuspended in 25 \uL\ of TLE solution and combined the two pools of cleaned 
ligation product. 

In order to be able to detect and remove PCR duplicates, I performed a single   
cycle of PCR with the iTru5-8N primer which adds a random 8 nucleotide barcode to 
each library construct.  
For each plate, I prepared four PCR reactions with a total volume of 
50 \uL\ containing 1X Kapa Hifi Buffer (Kapa Biosystems, USA; Kapa),
0.3 \uM\ iTru5-8N Primer, 0.3 mM dNTP, 1 unit Kapa HiFi DNA Polymerase,
10 \uL\ of purified ligation product, and nuclease free water.
I ran reactions through a single cycle of PCR on a thermocycler at 98\degree C for 2 min, 
60\degree C for 30 s, and 72\degree C for 5 min. 
I pooled all of the PCR products for a plate into a single tube and purified the
libraries with a 2X volume of SpeedBead solution as described before and 
resuspended in 25 \uL\ TLE.
I added the remaining adapter and index sequences which were unique to each plate with four PCR
reactions with a total volume of 50 \uL\ containing 1X Kapa Hifi (Kapa),
0.3 \uM\ iTru7 Primer, 0.3 \uM\ P5 Primer, 0.3 mM dNTP, 1 unit of Kapa Hifi DNA Polymerase (Kapa),
10 \uL\ purified iTru5-8N PCR product, and nuclease free water.
I ran reactions on a thermocycler with an initial denaturation at 98\degree C for 2 min, 
followed by 6 cycles of 98\degree C for 20 s, 60\degree C for 15 s, 72\degree C 
for 30 s and a final extension of 72\degree C for 5 min.
I pooled all of the PCR products for a plate into a single tube and purified the
product with a 2X volume of SpeedBead solution as described before and 
resuspended in 45 \uL\ TLE.

I size selected the library DNA from each plate in the range of 450-650 base pairs using
a BluePippin (Sage Science, USA) with a 1.5\% dye free gel with internal R2 standards. 
To increase the final DNA concentrations I prepared four PCR reactions for each 
plate with 1X Kapa Hifi (Kapa), 0.3 \uM\ P5 Primer, 0.3 \uM\ P7 Primer, 0.3 mM dNTP, 
1 unit of Kapa HiFi DNA Polymerase (Kapa), 10 \uL\ size selected DNA, and 
nuclease free water and used the same thermocycling conditions as the previous
(P5-iTru7) amplification.
I pooled all of the PCR products for a plate into a single tube and purified 
the product with a 2X volume of SpeedBead solution as before and resuspended in 20 \uL\ TLE. 
I quantified the DNA concentration for each plate with a Qubit fluorometer 
(Life Technologies, USA) then pooled each plate in equimolar amounts relative 
to the number of samples on the plate and diluted the pooled DNA to 5 nM with
TLE solution. 
The pooled libraries were pooled with other projects and sequenced on an Illumina 
HiSeqX by Novogene (China) to obtain paired end, 150 base pair sequences. 

\subsection{Data Processing}
I demultiplexed the iTru7 indexes using the \processradtags command from 
\stacks v2.6.4 \parencites{rochette2019} and allowed for two mismatches for rescuing reads.
To remove PCR duplicates, I used the \clonefilter command from \stacks.
I demultiplexed inline sample barcodes, trimmed adapter sequence, and filtered 
reads with low quality scores as well as reads with any uncalled bases using the  
\processradtags command again and allowed for the rescue of restriction site sequence 
as well as barcodes with up to two mismatches.  
I built alignments from the processed reads using the \stacks pipeline. 
I allowed for 14 mismatches between alleles within, as well as between individuals
(M and n parameters). This is equivalent to a sequence similarity threshold of   
90\% for the 140 bp length of reads post trimming. 
I also allowed for up to 7 gaps between alleles within and between individuals.
I used the \populations command from \stacks to filter loci missing in more than   
5\% of individuals, filter all sites with minor allele counts less than 3, filter 
any individuals with more than 90\% missing loci, and randomly sample a single
SNP from each locus.

\subsection{Genetic Clustering \& Ancestry Proportions}
To cluster individuals and characterize patterns of genetic differentiation and 
admixture between clusters, I used the Bayesian inference program 
\structure v2.3.4 \parencite{pritchard2000} with \structure's 
admixture model which returns an estimate of ancestry proportions for each sample. 
To evaluate the assumption that samples are best modeled as inheriting their    
genetic variation just two groups corresponding to the species identification  
made in the field, I ran \structure under four different models, each with a different number of 
assumed clusters of individuals (K parameter) ranging from 1 to 4. For each value of K, I 
ran 20 iterations for 100,000 total steps with the first 50,000 as burnin. 
I used the R package \pophelper v2.3.1 \parencite{francis2017} to combine
iterations for each value of K and to select the model producing the largest
$\Delta$K which is the the model that has the greatest increase in likelihood 
score from the model with one fewer populations as described by \parencite{evanno2005}.
I also examined genetic clustering and evidence of admixture using a non-parametric 
approach with a principal component analysis (PCA) implemented in the R package \adegenet v2.1.10 \parencite{jombart2008}. 
I visualized the relationship between the first principal component axis and the 
estimated admixture proportion for each individual to check for agreement  
between the parametric \structure analysis and the non-parametric PCA analysis.

\subsection{Genomic Cline Analysis}
To investigate patterns of introgression across the hybrid zone I     
used the bayesian genomic cline inference tool \bgc v1.03 \parencite{gompert2012} 
to infer parameters under a genomic cline model.
Explain a bit how \bgc works??? \ldots
I classified a sample as being admixed if it had an inferred admixture proportion
of <95\% for one species under the model with a K of two in 
the \structure analysis.
I used \vcftools vXX.XX.XX to filter all non-biallelic sites from the the VCF file 
produced by the \populations command in \stacks. 
I converted the VCF formatted data into the \bgc format using \bgcutils v0.1.0,  
a \python package that I developed for this project \url{github.com/kerrycobb/bgc_utils}.
I ran \bgc with 5 independent chains, each for 1,000,000 steps and sampling every 1000.
I visualized MCMC output, discarded samples from the posterior as burnin, combined the independent 
chains, summarized the posterior samples, and identified outlier loci with \bgcutils. 
A primary goal of \bgc analysis is to identify loci which have exceptional 
patterns of introgression. These loci, or loci in close linkage to them, are
expected to be enriched for genetic regions affected by selection due to 
reproductive incompatibility between the two species.
I identified loci with exceptional patterns of introgression using two approaches 
described by \cite{gompert2011}.
(1) If locus specific introgression differed from the genome-wide average which 
I will refer to as "excess ancestry" following \cite{gompert2011}.
More specifically, I classified a locus as having excess ancestry 
if the 90\% highest posterior density interval (HPDI) for the alpha or beta 
parameter did not cover zero.
(2) If locus specific introgression is statistically unlikely relative to the 
genome-wide distribution of locus specific introgression which I will refer to 
as "outliers" following \cite{gompert2011}.
I classified a locus as an outlier if the median of the posterior sample for the 
$\alpha$ or $\beta$ parameters for a locus were not contained 
the interval from 0.05 to 0.95 of the probability density functions $Normal(0, \tau_\alpha)$  
or $Normal(0, \tau_\beta)$ respectively, where $\tau_\alpha$ and $\tau_\beta$ are  
the median values from the posterior sample for the conditional random effect 
priors on $\tau_\alpha$ and $\tau_\beta$.
These conditional priors describe the genome-wide variation of locus specific 
$\alpha$ and $\beta$.
I further classified outlier $\alpha$ parameter estimates for a locus based on whether the 
median of the posterior sample was positive or negative. 
Positive estimates of $\alpha$ mean there is a greater probability of \amer 
ancestry in individuals at the locus relative to their hybrid index whereas negative 
estimates of $\alpha$ mean there is a greater probability of \terr ancestry. 

\subsection{Genetic differentiation and Introgression}
To test for a relationship between patterns of introgression and genetic divergence 
I used \vcftools to calculate the \cite{weir1984} \fst between 
each species using only the samples inferred through the \structure analysis to have 
>95\% ancestry for one species under the model with a K of two \parencite{danecek2011}. 
The Weir and Cockerham \fst is calculated per site and I calculated the per site 
\fst for the same sites as those used in the \bgc analysis. 
To determine if patterns of introgression are correlated with population differentiation
a performed a Pearson Correlation to test if \fst correlates with either the
$\alpha$ or $\beta$ parameters. I ran the correlation test with the absolute  
value of the median of the posterior sample for the $\alpha$ parameter and 
the median of the posterior sample for the $\beta$ parameter.
I binned loci based on their status as $\alpha$ parameter outliers. I categorized  
loci as being $\alpha$ outliers greater expected \amer
ancestry, $\alpha$ outliers with greater expected \terr ancestry, and estimates 
of $\alpha$ that are not outliers.
I performed a Kruskal-Wallis test using \scipy v1.10.1 to test whether there were 
significant differences in values of \fst at each locus between these groups \parencite{scipy}. 
I then performed Mann-Whitney tests between all pairs of groups using \scikitpost to test which 
groups differ significantly from each other \url{github.com/maximtrp/scikit-posthocs}.


\section{Results}
\subsection{Sampling and Data Processing}
I prepared reduced-representation sequencing libraries from 173 samples collected 
for this study (\cref{table:collectedHyb}) and 19 samples available from existing 
collections (\cref{table:loanedHyb})). 
The \stacks pipeline assembled reads into 432,336 loci with a mean length of 253.31 bp.
Prior to filtering the mean coverage per sample was 32X.
After filtering loci missing from greater than 5\% of samples, filtering sites with   
minor allele counts less than 3, filtering individuals with greater than 90\% 
missing loci, and randomly sampling a single SNP from each locus, 1194 sites
remained and 43 samples were excluded from further analyses leaving a total of 149. 
For the included samples, 56 had been identified as most closely resembling \amer 
and 93 had been identified as most closely resembling \terr. 

\subsection{Genetic Clustering \& Ancestry Proportions}
A visual inspection of the \structure results shows that each iteration with  
same value for K converged on very similar results (\cref{fig:structureIter}). 
The \structure model with the largest $\Delta$K was the model with a K of two (\cref{fig:deltak}).
Furthermore, individuals are inferred as having ancestry derived largely 
from only two ancestral groups even for K values of three and four. For these values
of K, only a small amount of ancestry is attributed to the third or fourth 
ancestral groups for any individual sample (\cref{fig:allk})
Using a 95\% estimated ancestry proportion as a cutoff for considering individuals to have
pure ancestry, 36 samples were classified as pure \amer, 75 as pure \terr, and  
38 as being admixed. 
The proportions of admixture among the samples shows a clear gradient between 0 
and 1 which is consistent with many individuals being the product of advanced 
generation hybrids beyond the $F_1$ generation.
The transition of admixture proportions from one species to the other 
increase with distance from the locations of pure individuals with proportions 
closest to 0.5 being found in the center of this transition (\cref{fig:hybrid-main}).

\subsection{Patterns of Introgression}
Visualization of the MCMC output with trace plots and histograms of each parameter 
indicated that each of the five chains run in \bgc converged on the same parameter 
space and that each chain quickly reached stationarity. 
I conservatively discarded the first 10\% of samples as burnin. 
The median of the posterior sample for $\alpha$ ranged from -0.525-0.494. 
The $\beta$ parameter was less variable and ranged from-0.158-0.220. 
I identified 16 loci with excess ancestry for the $\alpha$ parameter relative to 
the genome wide average; i.e., the 90\% HDPI does not cover 0. 
Of these, the median of the posterior sample for 5 of these loci was negative   
and for 11 loci was positive. Negative values represent a greater probability 
of \amer ancestry at a locus relative to the hybrid index whereas positive values 
represent a greater probability of \terr ancestry.
I did not identify any loci for which the estimates of $\beta$ were outliers
relative to the genome-wide average. 
I identified 116 loci as outliers for the $\alpha$ parameter relative to the
genome-wide distribution of locus specific introgression. 
Of these, the median of the posterior sample for 24 of these loci was negative
and for 92 loci was positive (\cref{fig:cline}).
I did not identify any loci for which the estimates of $\beta$ were outliers
relative to the genome-wide distribution of locus specific introgression. 
All 16 of the loci identified as having excess ancestry for the $\alpha$ parameter relative to 
the genome-wide average were also identified as outliers relative to the  
genome-wide distribution of locus specific introgression.

\subsection{Genomic Differentiation}
Genetic differentiation between \amer and \terr was highly variable (\cref{fig:hybrid-fst-box}). 
Locus-specific \fst between non-admixed \amer and \terr had a mean 
of 0.07. \fst values for 249 loci were 0. Only a single locus had fixed 
differences between species with an \fst of 1.0. 
There is little apparent relationship between $\alpha$ or $\beta$ and \fst 
except at that the highest $\alpha$ and $\beta$ estimates have non-zero \fst estimates (\cref{fig:hybrid-fst-relationship}).
The Pearson correlation test estimates a weak correlation between $\alpha$ and
\fst (r=0.29, p=1.62e-23) a week correlation between $\beta$ and \fst 
(r=0.32, $p=\num{8.28e-30}$ ). 
The result of the Kruskal-Wallis test are consistent with there being significant 
differences between the \fst values of loci with outlier $\alpha$ estimates
and non-outlier $\alpha$ estimates on average ($p=\num{1.32e-40}$) (\cref{fig:hybrid-fst-box}).
The results of the post hoc pairwise Mann-Whitney tests are consistent with 
both categories of loci with outlier $\alpha$ estimates  having greater \fst 
values on average than the non-outlier estimates of  $\alpha$.
The difference between non-outlier loci and loci with greater probability of
\amer ancestry was slightly higher ($p=\num{2.72e-38}$) than the difference between 
non-outlier loci and loci with greater \terr ancestry ($p=\num{8.16e-6}$).


% To understand how locus-specific patterns of genetic differentiation in allopatry 
% may impact the process of hybridization, we compared FST distributions between 
% allopatric parentals across each locus category (neutrally introgressing, 
% C. o. concolor excess ancestry, and C. v. viridis excess ancestry). 
% Excess ancestry loci from either parental lineage had significantly higher

% overall FST values than neutrally introgressing markers (ANOVA; P < 2.0 × 10−16) 
% and loci that had C. o. concolor ancestry (α<0) had higher FST on average than 
% loci with excess C. v. viridis ancestry (α>0, Welch’s two-sample t-test, effect 
% size 0.6; P < 2 × 10−16; Figs. 4, S11). We also found a low but significant 
% positive relationship between locus-specific genetic differentiation of parental 
% populations and absolute values of the genomic cline center parameter (Pearson’s 
% correlation r = 0.04; P < 6.0 × 10−5; Fig. S11)

\section{Discussion}
\subsection{Evidence for ongoing hybridization}
% Incompatible combinations of alleles arise through the process of recombination
% happening from backcrossing \parencite{orr1996,barton1985}
% Even in the absence of the genetic data presented in this study, the contact zone 
% of \amer and \terr seems to bear the hallmarks of a "tension zone" where 
% species barriers are maintained by a balance between dispersal and selection 
% against individuals carrying certain hybrid genotypes \parencite{barton1985}.
% The ranges of \amer and \terr abut with an abrupt transition and no apparent overlap. 
% Furthermore, the boundary between these species forms a long, smooth arc from Louisianna 
% to Virginia with a position closely corresponding to the fall line, 
% a prominent physiographic transition between the Piedmont Plateau in the North 
% and the southern coastal plain in the South (\cref{fig:hybrid-main}).
% Tension zones are expected to correspond with natural features that reduce
% dispersal or reduce the density of \parencite{barton1979}.
% Such a sudden transition is difficult to explain except if it is not the result of 
% the processes characteristic of tension zones. 
% For there to be no mutually hospitable areas permitting some range overlap is 
% implausible without there being an extreme level of competition or extreme degree 
% of adaptation by each species to their respective environments.
% The similarity of male advertisement calls, overlap in spawning period, and
% laboratory crossing experiments demonstrating reproductive compatibility suggest
% barriers to hybridization are weak.
% Previous analysis of morphological variation by \cite{weatherby1982} 
% showing the presence of morphological intermediates across a large region of 
% central Alabama suggested that introgression is occurring.

% Detecting introgression 
With the genome-wide sequence data obtained in this study, I find evidence of
substantial gene flow across the hybrid zone of these two species.
The \structure analysis inferred 38 out of 149 samples as having a proportion of 
ancestry of at least 5\% of sites attributable to admixture (\cref{fig:hybrid-main}). 
The admixture proportions inferred in the \structure analysis range 
from 0.05\%-0.5\% which is consistent with hybrids being viable, fertile, and 
capable of backcrossing over multiple generations \citationNeeded (\cref{fig:hybrid-main}). 
When backcrossing occurs over multiple generations in combination with migration
of hybrid progeny and selection against introgressing alleles, a cline will form 
across the hybrid zone with introgressing alleles becoming more uncommon with
distance from the cline center \parencite{barton1985}.
The results of the \structure analysis are largely consistent with this.
Inferred admixture coefficients are highest at the center of the hybrid zone 
and decrease and approach zero with distance from the center (\cref{fig:hybrid-main}).

% Width of hybrid zone
Admixed samples were located quite far from the center of the hybrid zone. 
In fact samples with greater than 5\% admixture proportions are located all the
way at the Northeastern and Southwestern edges of the sampling area. 
The width of a hybrid zone is a product of the strength of selection for or against
introgression and the average dispersal distance of individuals within their 
reproductive lifespan \parencite{barton1985}.
\cite{breden1987} estimated that 27\% of individual \textit{A. fowleri} breed
at non-natal breeding ponds with some individuals dispersing at least as much 
as 2 km.
Female \amer can migrate at least 1 km between breeding sites and post-breeding
locations \parencite{forester2006}
Invasive cane toads (\textit{Rhinella marina}) in Australia are estimated to 
have expanded their range at a rate of 10-15 km per year shortly after their
introduction \parencite{urban2008}. 
The presence of samples with little to no admixture in close proximity to 
toads with high proportions of admixture shows that dispersal has an important  
roll in shaping the patterns of this hybrid zone.
Individuals would be expected to appear more like their neighbors if dispersal
rates and distances were very low.
It is also likely that this hybrid zone may be more appropriately described as 
a mosaic hybrid zone rather than a more simple tension zone \parencite{harrison1986}.
However, the sampling for this study or too sparse and irregular to definitively 
test this.
Another possibility is that some of this inferred admixture is the result of a 
statistical artifact or due to error. 
Some reassurance is provided by the result of the PCA which is largely consistent 
with the \structure results although it is possible that they could be affected 
by the same bias or error introduced at by data collection and processing (\cref{fig:hybrid-main}) \citationNeeded.

% Location of hybrid zone center
The tension zone model of hybrid zones predicts that location of hybrid zones
centers will be dependent on the effects of selection along with population 
density and natural dispersal barriers \parencite{barton1979}.
The \structure results show that in two areas, there is a clear transition from 
samples with primarily \amer ancestry to samples with primarily \terr ancestry 
corresponding with the locations of streams and rivers.
In the Northern part of the sampling area, transitions occur at the Coosa 
River and at Waxahatchee Creek (\cref{fig:hybrid-main}). 
In the Southern part, they occur at Sougahatchee Creek (\cref{fig:hybrid-main}).
Clearly these are not impassable boundaries as there has been introgression 
beyond them. 
However, they likely reduce dispersal and as a result that the center of the hybrid 
zone is caught in this location as described by \cite{barton1979}.

\subsection{Variability of introgression}
There are two primary parameters of interest in a genomic cline model that 
can be interpreted in the evolutionary context of hybrid zones.
The $\alpha$ parameter specifies the center of the cline and is dependent on the 
increase or decrease in the probability of locus-specific ancestry from one of 
the parental populations.  
The $\beta$ parameter specifies the rate of change in probability of ancestry 
along the genome-wide admixture gradient.  
Extreme estimates of these parameters may be associated with loci that cause  
reproductive incompatibility between hybridizing species. 
The Bayesian genomic cline analysis of the genome-wide data in this study yielded extreme 
estimates for $\alpha$ at some sites.
Sites were classified as having extreme values in two ways. 
First, sites could be classified as having excess ancestry if the HDPI does not cover
zero and is therefore extreme relative to the genome-wide average of cline 
parameter estimates.
Second, sites could be classified as being outliers if they are extreme relative  
to the genome-wide distribution of locus specific effects under the cline model.
A greater number of sites qualified as outliers for estimates of $\alpha$ than 
qualified as having excess ancestry.
There were 116 loci classified as outliers which make up 9.7\% of the total 
number of sites.
Of those, 16 were also classified as having excess ancestry making up 1.3\% of all sites.
This difference is consistent with other studies using both simulated and 
empirical data which typically find more outlier loci than excess ancestry loci \parencite{gompert2012,}. 
Both of these methods can produce false positives as these extreme values can
be produced solely by genetic drift rather than by by selection \parencite{gompert2012}. 
So not all sites with extreme estimates will be associated with incompatibility loci. 
The false positive rate is exacerbated when there are many loci with small 
effects on compatibility. 
However, these sites should be enriched for loci associated with modest to strong 
reproductive incompatibility and thus provide an upper estimate of the number of 
sites that are associated with these modest to strong barriers to gene flow \parencite{gompert2012}. 

None of the estimates for $\beta$ were classified as either outliers or as having excess ancestry.
Simulations have demonstrated that the $\alpha$ parameter is more impacted by  
selection against hybrid genotypes than the $\beta$ parameter \parencite{gompert2012a}. 
Other studies have also found no extreme estimates of $\beta$ \parencite{nikolakis2022,gompert2012a}\citationNeeded. 
One possible interpretation of the absence of extreme values of $\beta$ is that
selection is only strong enough to have a significant impact on $\alpha$ but it 
is not strong enough to have a large impact on $\beta$.
Unlike for $\alpha$, there is not a strong relationship between locally positive 
selection favoring introgressed geneotypes and $\beta$ \parencite{gompert2012a}.
Therefore, some of the extreme values for $\alpha$ could be due to adaptive 
introgression which does have much impact on estimates of $\beta$.
This is plausible given the large extent of introgression which is potentially 
due to adaptive introgression.
There is a negative relationship between $\beta$ and dispersal rate \parencite{gompert2012a}.
It is also plausible that high dispersal rates, rather than selection is the 
cause of lower $\beta$ values that do not reach the threshold to qualify as extreme.

% Asymmetry
Of the 9.7\% of sites that qualified as $\alpha$ outliers, a substantially larger 
proportion had positive values which represent greater \amer ancestry
than expected at those sites in admixed individuals. 
Negative $\alpha$ estimates represent a greater probability of \terr ancestry 
at a site in within admixed individuals.
Sites with positive outlier estimates for $\alpha$ made up 7.7\% of all sites 
whereas those with negative outlier estimates made up just 2\%.
This asymmetry suggests that introgression flows more in the direction of \amer 
than it does in the direction of \terr.
This result is consistent with a pattern evident upon visual inspection of the 
mapped \structure results.
Samples collected from sites adjacent to sites with admixed samples appear to 
have a greater proportion of \amer ancestry than \terr ancestry (\cref{fig:hybrid-main}). 
Taken together, these observations suggest that introgression at this hybrid 
zone is asymmetric \parencite{yang2020}.
Asymmetries in introgression can arise for multiple reasons.
There could differences in mate choice which make females of one species 
more selective than females of the other \parencite{baldassarre2014}. 
There can also be species differences in dispersal tendencies \citationNeeded.  
Reciprocal-cross differences in reproductive isolation, termed Darwin's Corollary,
are very common \parencite{turelli2007}.
If one of the sexes is more prone to dispersal, introgression will flow more 
freely in one direction that it would in the other. 
It is possible that this observation is just an artifact of sampling. 
Particularly if this is a highly mosaic hybrid zone.
However, many more samples with primarily \terr ancestry were collected than 
samples with primarily \amer ancestry. 




% A reference genome for these species would greatly enhance this analysis
% Sampled from what might be considered multiple populations which could be a problem
% Treated a fairly broad area as a single population.

% They could be affected by different processes. %Cite study showing spatial and temporal differences across a hybrid zone


% % Reference Genome
% A reference genome for these species would greatly enhance the explanatory power of
% this analysis allowing for an exploration of the genomic architecture of the   
% reproductive isolation between these lineages. Which regions of the genome are 
% involved in isolation. Are they more associated with sex chromosomes?
% The availability of vertebrate genomes is increasing rapidly so it will likely 
% be possible to map the outlier loci identified in this study to a reference genome
% for \amer or \terr to reveal patterns in the distribution of these putative 
% markers for genetic incompatibility. 
% % Mitochondria
% Mitochondrial data would also be valuable. Would allow for the detection of 
% mitonuclear incompatibilities that could be contributing to reproductive isolation.
% Could also be used to test for the effect of Haldane's rule which would predict  
% that females would be more affected by hybridization. The estimated cline
% for mitochondria would be expected to be much steeper \parencite{carling2008}.

% % Geographic cline
% With this study I have provided a characterization of this hybrid zone that can
% serve as a guide for future sampling. 
% Sampling along transects.
% Could then apply geographic cline methods
% Would give spatial insights.



\subsection{Relationship between introgression and differentiation}
Patterns of genetic differentiation and genomic introgression between
\amer and \terr are consistent with the hypothesis that regions of the genome
experiencing divergent selection also affect hybrid fitness.
As predicted, there is a positive association between locus specific estimates
of \fst and both the absolute value of the $\alpha$ and the $\beta$ parameter 
estimates.
Although this correlation supports the hypothesis that introgression outliers 
are linked to loci under selection, the association is only a modest one. 
Despite this, it is notable all of the outlier $\alpha$ estimates as well as the
highest $\beta$ estimates have non-zero \fst estimates. 
Whereas sites with lower $\alpha$ and $\beta$ estimates span the entire range 
from zero to one. 
This is consistent with expectations of secondary contact where not all loci 
that have undergone genomic divergence will necessarily result in 
reproductive isolation. 
A tighter coupling of divergence and resistance to gene flow would be expected
under a scenario of divergence with gene flow. 

% Caveat, Ancestry inference is dependent on allele frequency differences \parencite{gompert2017}




% Fst and Alpha are not independent so ANOVA assumptions may be violated
% Alpha outliers are consistent with directional selection?
% Few loci being outliers suggests not many parts of the genome are responsible 
% % Effects of selection, idiosyncracies
% With weak selection and low dispersal the false discovery rate will be higher \parencite{gompert2011}
% Outlier loci are still expected to be enriched for genetic regions under selection.
% False discovery rate when patterns of introgression are affected by epistasis
% is quite high. Not all regions affecting introgression may be among outliers.  
% Would be better to look a genome differentiation in the context of a reference genome
% Introgression outliers are potentially linked to loci under selection
% Imperfect association between divergence and introgression is consistent with 
% secondary contact. 
% If speciation happened with gene flow, a tight linkage between the two would be 
% expected.
% Ancestry inference is dependent on allele frequency differences \parencite{gompert2017}
% Association between divergence and introgression is hard to explain from neutral 
% processes and suggests that highly differentiated loci affect hybrid fitness
% depending on the genomic background \parencite{gompert2017}


% \subsection{Conclusion}
% This study shows that there is a significant amount of gene flow between 
% \amer and \terr. 
% Apparently gene flow is asymmetric.
% The results raise some interesting possibilities but this study cannot 
% definitively address them.
% Is there adaptive introgression? 
% Is there very high dispersal?
% Selection weak?
% Improved sampling, a quality reference genome, and studies of natural history 
% can shed light on these.
% Use geographic cline

% % Multiple populations, potentially differing dynamics
% One limitation of this study is samples included in analysis are sampled from 
% a fairly broad area and may be more appropriately treated as separate populations.
% They could be affected by different processes. %Cite study showing spatial and temporal differences across a hybrid zone
% This hybrid zone could serve as an excellent system to explore such things
% given the large extent. Many important environmental factors likely differ 
% across the hybrid zone.


\subsection{Conclusion}

In conclusion, the genome-wide sequence data analysis conducted in this study 
has provided compelling evidence of significant gene flow across the hybrid zone 
of two distinct species. The STRUCTURE analysis reveals that a substantial 
number of samples exhibit evidence of admixture, with the proportion of ancestry 
attributed to hybridization ranging from 0.05\% to 0.5\%. These findings suggest 
that hybrids are not only viable and fertile but also capable of backcrossing 
over multiple generations. Furthermore, the spatial distribution of admixture 
coefficients suggests the formation of a cline, with the highest levels of 
admixture at the hybrid zone's center gradually diminishing with distance. 
Notably, some samples with substantial admixture are located at the edges of the 
hybrid zone, underscoring the influence of dispersal on shaping hybrid patterns.
 While the tension zone model predicts hybrid zone centers based on factors like 
 selection, population density, and natural dispersal barriers, the study reveals 
 that transitions between species dominance align with the presence of streams 
 and rivers. Although the data identifies outlier loci associated with reproductive
 incompatibility, it also underscores the complexities of hybridization, suggesting 
 that not all divergent loci result in reproductive isolation. In summary, this 
 research provides valuable insights into the dynamics of hybrid zones and the 
 role of selection, dispersal, and gene flow in shaping the genetic landscape 
 of these two species.

