\section{Introduction}
% A complete understanding of the processes that lead to Haldane’s rule (HR)—the 
% near ubiquitous tendency for the sex with non-identical sex chromosomes (heterogametic sex) 
% to suffer more from reproductive incompatibility in the form of sterility or 
% inviability following hybridization—has yet to be fully realized. 
% An understanding of such a rigidly held phenomenon will be key for gaining a more 
% complete understanding of the processes that drive the origin of species which is 
% one of the most enduring pursuits in biology. There are several non-exclusive 
% processes hypothesized to explain HR which are supported by both theory and 
% empirical data. It is expected that there is interplay among these processes, 
% but some may not be relevant to all groups of organisms (Delph and Demuth, 2016). 
% The “dominance hypothesis” for example, which is the most prominent hypothesis 
% explaining HR is only applicable to organisms with differentiated sex chromosomes 
% (heteromorphic) (Delph and Demuth, 2016). Under the dominance hypothesis, 
% reproductive incompatibility is caused by an interaction between a recessive locus 
% on the sex chromosome that is possessed by both sexes (X or Z chromosome) and 
% another locus on a different chromosome (Turelli and Orr, 1995). This incompatibility 
% is only expressed in the heterogametic sex due to the presence of a single copy 
% of the locus and therefore, is unlikely to explain HR in species with identical 
% sex chromosomes (homomorphic) as there are two copies of every or nearly every 
% locus in both sexes (Turelli and Orr, 1995). Another leading hypothesis is the 
% “faster-male hypothesis” which posits that genes related to the reproductive 
% systems of males undergo more rapid evolution or are more susceptible to 
% disruption in hybrids (Delph and Demuth, 2016). The “faster-male hypothesis” 
% cannot explain HR in organisms in which females are the heterogametic sex (ZW). 
% Neither of these hypotheses predict that HR would apply to species with homomorphic
% ZW sex chromosomes, yet greater inviability and sterility in females (the heterogametic sex) 
% has been found in such species (Malone and Fontenot, 2008). None of these species 
% have been studied in detail to examine the contributions and nature of other 
% hypothesized processes behind Haldane’s rule in the absence of others that have 
% received more attention. One way that HR can test for is by taking advantage of
% hybrid zones which provide a useful opportunity to observe the interaction of 
% divergent gene pools allowing for the investigation of the underlying processes
% that cause reproductive incompatibility (Gompert et al., 2017).


% Hybrid zones are geographic areas where divergent lineages interbreed and can 
% lead to the exchange of genetic variation between the involved lineages (i.e. introgression). 
% This genetic exchange can serve as a source of adaptive genetic variation in 
% recipient lineages (Elgvin et al., 2017). But hybridization can be costly when 
% reproductive incompatibility results in the production sterile or unfit offspring 
% (Barton and Hewitt, 1985). In these cases, selection might favor individuals that 
% mate assortatively and do not produce unfit offspring which would further increase 
% divergence between the two lineages—particularly in traits that enhance prezygotic 
% isolation (Servedio, 2000).

% A powerful tool for studying the interaction of divergent genomes at hybrid zones 
% is the cline model which quantifies genetic variation across hybrid zones. There 
% are two classifications of cline models: (1) geographic clines and (2) genomic 
% clines. Geographic cline models quantify variation across a linear transect of 
% a hybrid zone (Barton and Hewitt, 1985). These models make the assumption that 
% allele frequencies change monotonically from one side of the zone to the other, 
% but this assumption is sometimes violated such as in the case of mosaic hybrid 
% zones (Harrison, 1986). Genomic clines are not spatially explicit and quantify 
% genetic variation along an admixture gradient (Barton and Gale, 1993). An 
% admixture gradient is the varying proportion of genomes from two hybridizing 
% lineages present in the recombinant offspring of multi-generation hybrids. 
% Genomic clines can be utilized in cases where there is no obvious spatial axis 
% (Gompert et al., 2017).

% Cline analysis can quantify locus-specific introgression relative to the genome 
% wide average. Outlier loci are regarded as putatively under strong selection for 
% or against introgression based on relative position to the genome wide average. 
% These estimates can provide insight into the number of loci that are involved in 
% reproductive incompatibility, the relative strength of selection among loci, as 
% well as the overall strength of incompatibility between hybridizing lineages 
% (Gompert et al., 2017). Cline analysis has also been used to identify putatively 
% adaptive loci that introgress more readily between hybridizing lineages (Chhatre et al., 2018). 
% It can also be used detect asymmetrical gene flow indicative of sex biased 
% dispersal or asymmetric effects of reproductive incompatibility or fitness 
% (Barton and Hewitt, 1985). Estimates of differential rates of introgression 
% are also useful for understanding the genomic architecture of incompatibility 
% by revealing the location of loci associated with reproductive incompatibility (Janoušek et al., 2015).

% Cline analysis can also be used to determine if the outcome of hybridization 
% between two species is consistent with the expectations of HR. The rate of 
% introgression of any loci uniparentally inherited from the heterogametic sex
% are expected to be reduced as the individuals from which these loci are inherited 
% will be affected by reduced viability and fertility when the hybridizing species 
% are affected by HR. This would affect the rate of introgression of loci such as 
% those on Y chromosomes, W chromosomes, mitochondrial genomes, or the chloroplasts 
% of many plants. Clines of uniparentally inherited loci would be the steeper and 
% narrower than the clines of loci from other parts of the genome.

% Toads in the family Bufonidae have homomorphic sex chromosomes and most have 
% been shown to have ZW sex determination (Brelsford et al., 2013). Laboratory 
% crosses involving 92 different species indicate that most species are affected 
% by HR but that some could be rare exceptions (Malone and Fontenot, 2008). A 
% number of hybrid zones are known or suspected to exist between toad species 
% (Green 1996). One hybrid zone suspected on the basis of morphological data 
% involves the American toad and Southern toad in central Alabama, (Weatherby, Craig A, 1982). 
% The ranges of these two species do not overlap but do meet at a long zone of 
% contact in the Southeastern United States corresponding with a prominent 
% geological transition known as the “fall line” (Shankman and Hart, 2010). 
% The fall line separates the Appalachian highlands to the North from the coastal 
% plain to the South (Shankman and Hart, 2010). This pattern of non-overlapping 
% distribution is consistent with the existence of a tension zone between the two 
% species (Key, 1968). Tension zones form where an equilibrium is reached between 
% selection and migration resulting in the maintenance of species boundaries despite 
% ongoing hybridization (Barton and Hewitt, 1985). Tension zones are expected to 
% correspond with a transition in ecological conditions and with areas where both 
% species occur at lower abundance (Barton and Hewitt, 1985). Laboratory crosses 
% between these two species have been shown to result in viable and fertile first 
% generation male and female hybrid offspring (Blair, 1963). Second generation 
% offspring that result from back crossing first generation offspring to the parent 
% species do suffer from a partial reduction in viability during development, but 
% it has not been determined if one sex suffers more from this or if either sex 
% suffers from infertility (Blair, 1963). If hybrid offspring fertility and viability 
% is not consistent with HR, it would make this one of the exceptionally rare examples 
% known making it an excellent target for comparative study to understand the mechanisms 
% that generate HR.

% Below I propose a study using genome wide data to investigate the consequences 
% of hybridization of American and Southern toads in central Alabama and assess 
% the applicability of HR in the hybrids of these two species. I will 1) test the 
% hypothesis that introgression occurs between American and Southern toads where 
% the ranges of these two species meet in central Alabama and 2) test the hypothesis 
% that hybrid females have reduced viability or sterility consistent with Haldane’s Rule.

% %%%%%%%%%%%%%%%%%%%%%%%%%%%
% % New stuff
% % Uncovering the processes that generate and sustain biological diversity has been a 
% % fundamental pursuit in the field of evolutionary biology since its inception.
% % Consequently, the phenomenon of natural hybridization between distinct evolutionary 
% % lineages has captured the attention of many evolutionary biologists as far back as 
% % Darwin and his contemporaries \parencite{mallet2008}.
% % In part because it has been viewed as being at odds with the generation 
% % and maintenance of species diversity 
% % \parencite{mallet2008}.
% % The phenomenon of natural hybridization between distinct evolutionary lineages has 
% % captured the attention of evolutionary biologists since the field's inception \parencite{mallet2008}.

% The phenomenon of natural hybridization between 

% However, it is now widely appreciated that hybridization is compatible ... and many hybrid
% zones are know the exist and are stable... \parencite{mallet2005}

% Define hybrid zone ...

% As genetic data have become accessible, more instances of hybrid zones have been identified. 
% % Furthermore, more and more instances of hybridization are becoming known.  

% % It is now appreciated that natural hybridization is quite widespread and there
% % exist many hybrid zones between species where hybridization is frequent, yet 
% % the distinct identities of each remains stable \parencite{mallet2005}.

% % Consequences of hybridization:
% While hybrid zones remain stable over long periods of time, they are not necessarily  
% without consequence.

% Hybird zones are stable over long periods of time \parencite{deraad2023}

% It is even conceivable that hybridization could erode the differences between 
% % some slightly divergent groups of organisms in limited circumstances  \citationNeeded.
% % This is gaining attention due to the impact of human activity. \citationNeeded 

% Genetic exchange across hybrid zones can be a source can serve as a source of adaptive genetic variation in 
% recipient lineages \parencite{whitney2010}. 


% But hybridization can be costly when 
% reproductive incompatibility results in the production sterile or unfit offspring 
% (Barton and Hewitt, 1985). In these cases, selection might favor individuals that 
% mate assortatively and do not produce unfit offspring which would further increase 
% divergence between the two lineages—particularly in traits that enhance prezygotic 
% isolation (Servedio, 2000).

% % Hybridization is a possible source of adaptive variation \citationNeeded.
% Hybridization could also lead to the evolution of greater differences between species
% % or populations \citationNeeded.

% % What hybrid zones can tell us:
% Hybrid zones have also come to be appreciated as a valuable source of 
% insight into the evolutionary processes that drive diversification and as a potential 
% process with important consequences in and of themselves \parencite{barton1985}.

% Hybrid zones provide an opportunity for investigating the interaction of 
% genomic elements from the genomes of divergent lineages under natural conditions 
% and without the need for large breeding experiments.

% This allows for the investigation of the genomic architecture of reproductive 
% incompatibility that prevents species from interbreeding and adaptive divergence.
% The number of regions in the genome and their distribution throughout the genome.

% Asymmetries in gene flow. 

% Types of sexual selection

% Identify incompatibility loci (mouse sterility loci) \parencite{turner2014}


% % % Clines
% % A common approach to studying hybrid zone is cline analysis whereby parameters  
% % of a cline function are fit to genetic or phenotypic data to model the migration
% % and selection within the hybrid zone.


% History and prevalence of natural hybridization in toads 
A prominent group of organisms in the literature on hybridization are the "true
toads" in the family Bufonidae. 
W.F. Blair and colleagues performed a remarkable 1,934 separate experimental crosses 
to quantify the degree of reproductive incompatibility between species pairs within
Bufonidae \parencite{blair1972,malone2008}.
These experiments demonstrated a high degree of compatibility between some closely 
related species pairs in which hybrids were capable of producing viable backcross 
or $F_2$ hybrid offspring \parencite{blair1963}.
Furthermore, numerous cases of natural hybridization among toad species have been 
reported, including at several apparent or clear hybrid zones 
\parencite{green1996,vanriemsdijk2023,colliard2010,weatherby1982}.
Despite the interest and appreciation for hybridization in Bufonidae, only a 
small amount of work has been done to understand patterns of introgression
within hybrid zones. 
A clinal pattern of admixture at 26 allozyme loci has been show within the 
\textit{Anaxyrus americnaus} X \textit{Anaxyrus hemiophrys} hybrid zone in 
Ontario, Canada\parencite{green1983}.
Almost no admixture was detected at 7 microsatellite loci within the \textit{Bufo siculus} X 
\textit{Bufo balearicus} hybrid zone in Sicily, Italy \parencite{colliard2010}.
The most comprehensive study of introgression within a Bufonidae hybrid zone found 
significant levels of genome wide admixture, fitting a clinal pattern, at two separate transects 
at either end of the \textit{Bufo bufo} x \textit{Bufo spinosus} hybrid zone
in Southern France \parencite{vanriemsdijk2023}.

% Toads as an appealing target for study of hybridization
It is clear that significant levels of admixture occur within some Bufonidae 
hybrid zones which could yield insights into the evolution of reproductive incompatibility. 
There are a few qualities that make these hybrid zones particularly attractive for further investigation.
One of these qualities is the ease with which the primary behavioral isolating mechanisms, 
spawning period and advertisement call, can be measured and quantified in order 
to understand the strength of prezygotic mating barriers and possible patterns 
consistent with reinforcement \parencite{cocroft1995,blair1974,kennedy1962}.
It has also been show that they can be readily bred in captivity \parencite{blair1972}. 
Many species produce thousands of offspring which are externally fertilized  
making a variety of embryological manipulations or observations quite easy \parencite{blair1972}.
Breeding can be induced hormonally or performed in vitro, facilitating the 
planning and scheduling of experiments \parencite{trudeau2010}.
Unlike many of the organisms which have undergone intensive study in the context 
of speciation such as \textit{Mus} and \textit{Drosophila}, most Bufonidae
have homomorphic sex chromosomes \parencite{blair1972}.  
Furthermore, there is evidence of sex chromosome turnovers within Bufonidae \parencite{dufresnes2020,stock2011}. 
How does this change evolution of reproductive incompatibility?
These qualities along with the near global distribution of a large 642 species  
radiation present an excellent opportunity to further our understanding of the 
evolution of reproductive incompatibility \parencite{amphibiaweb2023} .

% Southern toad american toad hybrid zone
One suspected hybrid zone that has not been investigated with genetic data is 
between \amer and \terr in the Southern United States.
The ranges of these species meet but do not overlap at a long contact zone which
closely corresponds with a prominent physiographic feature known as the "fall line" \parencite{powell2016,mount1975}. 
The Fall line is the boundary between the Southern coastal plain to the South from 
the Appalachian Highlands to the North \parencite{shankman2007}.
These regions differ in their underlying geology, topography, and elevation \parencite{shankman2007}.
The distribution of \terr is restricted to the coastal plain extending from   
the Mississippi River in the West to Virginia in the East \cref{fig:hybrid-main}.
The distribution of the American Toad encompasses nearly all of the Eastern North 
American with the exception of the Southern coastal plain \cref{fig:hybrid-main}.
The two species have slight differences in male advertisement call and in 
morphological appearance \parencite{cocroft1995,weatherby1982}. 
They also differ slightly in the timing of their spawn \parencite{mount1975} 
However, there is significant overlap in the spawning period and male Bufondidae 
are famously indiscriminate in their choice of mates \parencite{dordevic2014,weatherby1982}.
Analysis of morphological variation in central Alabama has suggested there is 
introgression between them \parencite{weatherby1982}.
In this study, I investigate genome wide patterns of introgression along a portion
of the hybrid zone between \amer and \terr in central Alabama. 
I specifically address three questions.
1) Does introgression occur between \amer and \terr?
2) At what distance does introgression occur within the hybrid zone? 
3) Do patterns of introgression correlate with the level of population differentiation
of loci?

% % Are locus-specific measures of genetic differentiation positively correlated
% % with extreme introgression.?
% % locus-specific measures of genetic differentiation (a metric of divergent selection) 
% % will be positively correlated with extreme genomic introgression (a metric of selection in hybrids)


\section{Methods}
\subsection{Sampling and DNA Isolation}
I collected genetic samples from \textit{A. americanus} and \textit{A. terrestris}
by driving roads during rainy nights between 2017 and 2020
in an a region of central Alabama where hybridization has previously been
inferred from the presence of morphological intermediates \parencite{weatherby1982}. 
I euthanized individuals with immersion in buffered MS-222.
I removed liver and/or toes and preserved them in 100\% ethanol and fixed 
specimens with XXX M (ask David how he makes Formalin) Formalin solution.
Genetic samples and formalin fixed specimens were deposited in the Auburn Museum of Natural History.
Additional samples were also provided by museums (see Table 1).

I isolated DNA by lysing a small piece of liver or toe approximately 
the size of a grain of rice in 300 \uL\ of a solution of 10mM Tris-HCL, 10mM EDTA, 
1\% SDS (w/v), and nuclease free water along with 6 mg Proteinase K and 
incubating for 4-16 hours at 55\degree C.  
To purify the DNA and separate it from the lysis product, I mixed the lysis 
product with a 2X volume of SPRI bead solution containing 1 mM EDTA,  
10 mM Tris-HCl, 1 M NaCl, 0.275\% Tween-20 (v/v), 18\% PEG 8000 (w/v), 
2\% Sera-Mag SpeedBeads (GE Healthcare PN 65152105050250) (v/v), and nuclease free water.
I then incubated the samples at room temperature for 5 minutes, placed the 
beads on a magnetic rack, and discarded the supernatant once the beads had collected
on the side of the tube.  
I then performed two ethanol washes by adding 1 mL of 70\% ETOH to the beads
while still placed in the magnet stand and allowing it to stand for 5 minutes
before removing and discarding the ethanol. 
After removing all ethanol from the second wash, I removed the tube from the magnet 
stand and allowed the sample to dry for 1 minute before thoroughly mixing the beads with 100 \uL\ of 
TLE solution containing 10 mM Tris-HCL, 0.1 mm EDTA, and nuclease free water.
After allowing the bead mixture to stand at room temperature for 5 minutes I returned
the beads to the magnet stand, collected the TLE solution, and discarded the beads. 
I quantified DNA in the TLE solution with a Qubit
fluorometer (Life Technologies, USA) and diluted samples with additional TLE solution to 
bring the concentration to 20 ng/\uL.

\subsection{RADseq Library Preparation}
I prepared RADseq libraries using the 2RAD approach developed by \cite{bayona-vasquez2019}. 
On 96 well plates, I ligated 100 ng of sample DNA in 15 \uL\ of a solution with 
1X CutSmart Buffer (New England Biolabs, USA; NEB), 10 units of XbaI,
10 units of EcoRI, 0.33 \uM\ XbaI compatible adapter, 0.33 \uM\ EcoRI compatible adapter,
and nuclease free water with a 1 hour incubation at 37\degree C. 
I then immediately added 5 \uL\ of a solution with 1X Ligase Buffer (NEB),
0.75 mM ATP (NEB), 100 units DNA Ligase (NEB), and nuclease free water 
and incubated at 22\degree C for 20 min and 37\degree C for 10 min for two cycles, 
followed by 80\degree C for 20 min to stop enzyme activity.
For each 96 well plate, I pooled 10 \uL\ of each sample and split this pool 
equally between two microcentrifuge tubes.
I purified each pool of libraries with a 1X volume of SpeedBead solution followed 
by two ethanol washes as described in the previous section except that the DNA 
was resuspended in 25 \uL\ of TLE solution and combined the two pools of cleaned 
ligation product. 

In order to be able to detect and remove PCR duplicates, I performed a single   
cycle of PCR with the iTru5-8N primer which adds a random 8 nucleotide barcode to 
each library construct.  
For each plate, I prepared four PCR reactions with a total volume of 
50 \uL\ containing 1X Kapa Hifi Buffer (Kapa Biosystems, USA; Kapa),
0.3 \uM\ iTru5-8N Primer, 0.3 mM dNTP, 1 unit Kapa HiFi DNA Polymerase,
10 \uL\ of purified ligation product, and nuclease free water.
I ran reactions through a single cycle of PCR on a thermocycler at 98\degree C for 2 min, 
60\degree C for 30 s, and 72\degree C for 5 min. 
I pooled all of the PCR products for a plate into a single tube and purified the
libraries with a 2X volume of SpeedBead solution as described before and 
resuspended in 25 \uL\ TLE.
I added the remaining adapter and index sequences which were unique to each plate with four PCR
reactions with a total volume of 50 \uL\ containing 1X Kapa Hifi (Kapa),
0.3 \uM\ iTru7 Primer, 0.3 \uM\ P5 Primer, 0.3 mM dNTP, 1 unit of Kapa Hifi DNA Polymerase (Kapa),
10 \uL\ purified iTru5-8N PCR product, and nuclease free water.
I ran reactions on a thermocycler with an initial denaturation at 98\degree C for 2 min, 
followed by 6 cycles of 98\degree C for 20 s, 60\degree C for 15 s, 72\degree C 
for 30 s and a final extension of 72\degree C for 5 min.
I pooled all of the PCR products for a plate into a single tube and purified the
product with a 2X volume of SpeedBead solution as described before and 
resuspended in 45 \uL\ TLE.

I size selected the library DNA from each plate in the range of 450-650 base pairs using
a BluePippin (Sage Science, USA) with a 1.5\% dye free gel with internal R2 standards. 
To increase the final DNA concentrations I prepared four PCR reactions for each 
plate with 1X Kapa Hifi (Kapa), 0.3 \uM\ P5 Primer, 0.3 \uM\ P7 Primer, 0.3 mM dNTP, 
1 unit of Kapa HiFi DNA Polymerase (Kapa), 10 \uL\ size selected DNA, and 
nuclease free water and used the same thermocycling conditions as the previous
(P5-iTru7) amplification.
I pooled all of the PCR products for a plate into a single tube and purified 
the product with a 2X volume of SpeedBead solution as before and resuspended in 20 \uL\ TLE. 
I quantified the DNA concentration for each plate with a Qubit fluorometer 
(Life Technologies, USA) then pooled each plate in equimolar amounts relative 
to the number of samples on the plate and diluted the pooled DNA to 5 nM with
TLE solution. 
The pooled libraries were pooled with other projects and sequenced on an Illumina 
HiSeqX by Novogene (China) to obtain paired end, 150 base pair sequences. 

\subsection{Data Processing}
I demultiplexed the iTru7 indexes using the \processradtags command from 
\stacks v2.6.4 \parencites{rochette2019} and allowed for two mismatches for rescuing reads.
To remove PCR duplicates, I used the \clonefilter command from \stacks.
I demultiplexed inline sample barcodes, trimmed adapter sequence, and filtered 
reads with low quality scores as well as reads with any uncalled bases using the  
\processradtags command again and allowed for the rescue of restriction site sequence 
as well as barcodes with up to two mismatches.  
I built alignments from the processed reads using the \stacks pipeline. 
I allowed for 14 mismatches between alleles within, as well as between individuals
(M and n parameters). This is equivalent to a sequence similarity threshold of   
90\% for the 140 bp length of reads post trimming. 
I also allowed for up to 7 gaps between alleles within and between individuals.
I used the \populations command from \stacks to filter loci missing in more than   
5\% of individuals, filter all sites with minor allele counts less than 3, filter 
any individuals with more than 90\% missing loci, and randomly sample a single
SNP from each locus.

\subsection{Genetic Clustering \& Ancestry Proportions}
To cluster individuals and characterize patterns of genetic differentiation and 
admixture between clusters, I used the Bayesian inference program 
\structure v2.3.4 \parencite{pritchard2000} with \structure's 
admixture model which returns an estimate of ancestry proportions for each sample. 
To evaluate the assumption that samples are best modeled as inheriting their    
genetic variation just two groups corresponding to the species identification  
made in the field, I ran \structure under four different models, each with a different number of 
assumed clusters of individuals (K parameter) ranging from 1 to 4. For each value of K, I 
ran 20 iterations for 100,000 total steps with the first 50,000 as burnin. 
I used the R package \pophelper v2.3.1 \parencite{francis2017} to combine
iterations for each value of K and to select the model producing the largest
$\Delta$K which is the the model that has the greatest increase in likelihood 
score from the model with one fewer populations as described by \parencite{evanno2005}.
I also examined genetic clustering and evidence of admixture using a non-parametric 
approach with a principal component analysis (PCA) implemented in the R package \adegenet v2.1.10 \parencite{jombart2008}. 
I visualized the relationship between the first principal component axis and the 
estimated admixture proportion for each individual to check for agreement  
between the parametric \structure analysis and the non-parametric PCA analysis.

\subsection{Genomic Cline Analysis}
To investigate patterns of introgression across the hybrid zone I     
used the bayesian genomic cline inference tool \bgc v1.03 \parencite{gompert2012} 
to infer parameters under a genomic cline model.
I classified a sample as being admixed if it had an inferred admixture proportion
of <95\% for one species under the model with a K of two in 
the \structure analysis.
I used \vcftools vXX.XX.XX to filter all non-biallelic sites from the the VCF file 
produced by the \populations program. 
I converted the VCF formatted data into the \bgc format using \bgcutils v0.1.0 
\url{https://github.com/kerrycobb/bgc_utils}.
I ran \bgc with 5 independent chains, each for 1,000,000 steps and sampling every 1000.
I discarded the first 10\% of samples from the posterior, combined the independent 
chains, summarized the posterior samples, and identified outlier loci with \bgcutils. 
I classified loci as outliers and therefore expected to be enriched for 
incompatibility loci with two different approaches, (1) if locus specific 
introgression differed from the genome-wide average and (2) if locus specific 
introgression is statistically unlikely relative to the genome-wide distribution
of locus specific introgression. 
More specifically, in the first approach I classified a locus as an outlier 
if the 90\% highest posterior density interval (HPDI) for the alpha or beta 
parameter did not cover zero.
In the second approach, I classified a locus as an outlier if the median of the  
posterior sample for the $\alpha$ or $\beta$ parameters for a locus were not contained 
the interval from 0.05 to 0.95 of the probability density functions $Normal(0, \tau_\alpha)$  
or $Normal(0, \tau_\beta)$ respectively, where $\tau_\alpha$ and $\tau_\beta$ are  
the median values from the posterior sample for the conditional random effect 
priors on $\tau_\alpha$ and $\tau_\beta$ which can be thought of as the genome-wide
distribution for the cline parameters \parencite{gompert2011}.

\subsection{Genetic differentiation and Introgression}
To measure genetic differentiation I used \vcftools to calculate the \cite{weir1984} \fst between 
each species with only samples inferred through the \structure analysis to have 
>95\% for one species under the model with a K of two \parencite{danecek2011}. 
The Weir and Cockerham \fst is calculated per site and I calculated the per site 
\fst for the same sites as those used in the \bgc analysis. 
To determine if the rate of introgression is correlated with population differentiation
I binned loci based on their status as outliers using the 
whether they were determined to have an excess of \amer 
ancestry (negative $\alpha$ outlier) 



\section{Results}
\subsection{Sampling and Data Processing}
I prepared reduced-representation sequencing libraries from 173 samples collected 
for this study (\cref{table:collectedHyb}) and 19 samples available from existing 
collections (\cref{table:loanedHyb})). 
\stacks assembled reads into 432,336 loci with a mean length of 253.31 bp.
Prior to filtering the mean coverage per sample was 32X.
After filtering loci missing from greater than 5\% of samples, filtering sites with   
minor allele counts less than 3, filtering individuals with greater than 90\% 
missing loci, and randomly sampling a single SNP from each locus, 1194 sites
remained.
After assembly and filtering, 43 samples were excluded from analyses leaving a total of 149. 
For the included samples, 56 had been identified as most closely resembling \amer 
and 93 had been identified as most closely resembling \terr. 

\subsection{Genetic Clustering \& Ancestry Proportions}
A visual inspection of the \structure results shows that each iteration with  
same value for K converged on very similar results (\cref{fig:structureIter}). 
The \structure model with the largest $\Delta$K was the model with a K of two \cref{fig:deltak}.
Furthermore, individuals are inferred as having ancestry derived largely 
from only two ancestral groups even for K values of three and four. For these values
of K, only a small amount of ancestry is attributed to the third or fourth 
ancestral groups for any individual sample. \cref{fig:allk} 
Using a 95\% estimated ancestry proportion as a cutoff for considering individuals to have
pure ancestry, 36 samples were classified as pure \amer, 75 as pure \terr, and  
38 as being admixed. 
The proportions of admixture among the samples shows a clear gradient between 0 
and 1 which is consistent with many individuals being the product of advanced 
generation hybrids beyond the $F_1$ generation.
The transition of admixture proportions from one species to the other 
increase with distance from the locations of pure individuals with proportions 
closest to 0.5 being found in the center of this transition \cref{fig:hybrid-main}.

% \subsection{Genomic Cline \& Genetic Divergence}
% Mean \fst is 1 
% One locus fixed between species




% \section{Discussion}

% Alpha outliers are consistent with directional selection?

% Few loci being outliers suggests not many parts of the genome are responsible 



% % Results consistent with multiple generations of backcrossing

% % Effects of selection, idiosyncracies
% With weak selection and low dispersal the false discovery rate will be higher \parencite{gompert2011}
% Outlier loci are still expected to be enriched for genetic regions under selection.
% False discovery rate when patterns of introgression are affected by epistasis
% is quite high. Not all regions affecting introgression may be among outliers.  


% % Multiple populations, potentially differing dynamics
% One limitation of this study is samples included in analysis are sampled from 
% a fairly broad area and may be more appropriately treated as separate populations.
% They could be affected by different processes. %Cite study showing spatial and temporal differences across a hybrid zone
% This hybrid zone could serve as an excellent system to explore such things
% given the large extent. Many important environmental factors likely differ 
% across the hybrid zone.


% % Reference Genome
% A reference genome for these species would greatly enhance the explanatory power of
% this analysis allowing for an exploration of the genomic architecture of the   
% reproductive isolation between these lineages. Which regions of the genome are 
% involved in isolation. Are they more associated with sex chromosomes?
% The availability of vertebrate genomes is increasing rapidly so it will likely 
% be possible to map the outlier loci identified in this study to a reference genome
% for \amer or \terr to reveal patterns in the distribution of these putative 
% markers for genetic incompatibility. 


% % Mitochondria
% Mitochondrial data would also be valuable. Would allow for the detection of 
% mitonuclear incompatibilities that could be contributing to reproductive isolation.
% Could also be used to test for the effect of Haldane's rule which would predict  
% that females would be more affected by hybridization. The estimated cline
% for mitochondria would be expected to be much steeper \parencite{carling2008}.

% % Geographic cline
% With this study I have provided a characterization of this hybrid zone that can
% serve as a guide for future sampling. 
% Sampling along transects.
% Could then apply geographic cline methods
% Would give spatial insights.



